 <!doctype html><html lang="en" class="no-js"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width,initial-scale=1"><meta http-equiv="x-ua-compatible" content="ie=edge"><meta name="description" content="Sherlock Cluster documentation"><link rel="canonical" href="https://www.sherlock.stanford.edu/docs/software/_list/"><meta name="author" content="Kilian Cavalotti"><meta name="lang:clipboard.copy" content="Copy to clipboard"><meta name="lang:clipboard.copied" content="Copied to clipboard"><meta name="lang:search.language" content="en"><meta name="lang:search.pipeline.stopwords" content="True"><meta name="lang:search.pipeline.trimmer" content="True"><meta name="lang:search.result.none" content="No matching documents"><meta name="lang:search.result.one" content="1 matching document"><meta name="lang:search.result.other" content="# matching documents"><meta name="lang:search.tokenizer" content="[\s\_\-\.]+"><link rel="shortcut icon" href="../../../img/logo_small.png"><meta name="generator" content="mkdocs-1.0.4, mkdocs-material-4.3.1"><title> list - Sherlock</title><link rel="stylesheet" href="../../../assets/stylesheets/application.4031d38b.css"><link rel="stylesheet" href="../../../assets/stylesheets/application-palette.fe74ad26.css"><meta name="theme-color" content=""><script src="../../../assets/javascripts/modernizr.74668098.js"></script><link href="https://fonts.gstatic.com" rel="preconnect" crossorigin><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700|Roboto+Mono&display=swap"><style>body,input{font-family:"Roboto","Helvetica Neue",Helvetica,Arial,sans-serif}code,kbd,pre{font-family:"Roboto Mono","Courier New",Courier,monospace}</style><link rel="stylesheet" href="../../../assets/fonts/material-icons.css"><link rel="manifest" href="../../../manifest.webmanifest"><link rel="stylesheet" href="../../.."><script>window.ga = window.ga || function() {
    (ga.q = ga.q || []).push(arguments)
  }
  ga.l = +new Date
  /* Setup integration and send page view */
  ga("create", "UA-68034405-2", "auto")
  ga("set", "anonymizeIp", true)
  ga("send", "pageview")
  /* Register handler to log search on blur */
  document.addEventListener("DOMContentLoaded", () => {
    if (document.forms.search) {
      var query = document.forms.search.query
      query.addEventListener("blur", function() {
        if (this.value) {
          var path = document.location.pathname;
          ga("send", "pageview", path + "?q=" + this.value)
        }
      })
    }
  })</script><script async src="https://www.google-analytics.com/analytics.js"></script></head><body dir="ltr" data-md-color-primary="stanford" data-md-color-accent="stanford"><svg class="md-svg"><defs><svg xmlns="http://www.w3.org/2000/svg" width="416" height="448" viewBox="0 0 416 448" id="__github"><path fill="currentColor" d="M160 304q0 10-3.125 20.5t-10.75 19T128 352t-18.125-8.5-10.75-19T96 304t3.125-20.5 10.75-19T128 256t18.125 8.5 10.75 19T160 304zm160 0q0 10-3.125 20.5t-10.75 19T288 352t-18.125-8.5-10.75-19T256 304t3.125-20.5 10.75-19T288 256t18.125 8.5 10.75 19T320 304zm40 0q0-30-17.25-51T296 232q-10.25 0-48.75 5.25Q229.5 240 208 240t-39.25-2.75Q130.75 232 120 232q-29.5 0-46.75 21T56 304q0 22 8 38.375t20.25 25.75 30.5 15 35 7.375 37.25 1.75h42q20.5 0 37.25-1.75t35-7.375 30.5-15 20.25-25.75T360 304zm56-44q0 51.75-15.25 82.75-9.5 19.25-26.375 33.25t-35.25 21.5-42.5 11.875-42.875 5.5T212 416q-19.5 0-35.5-.75t-36.875-3.125-38.125-7.5-34.25-12.875T37 371.5t-21.5-28.75Q0 312 0 260q0-59.25 34-99-6.75-20.5-6.75-42.5 0-29 12.75-54.5 27 0 47.5 9.875t47.25 30.875Q171.5 96 212 96q37 0 70 8 26.25-20.5 46.75-30.25T376 64q12.75 25.5 12.75 54.5 0 21.75-6.75 42 34 40 34 99.5z"/></svg></defs></svg> <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off"> <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off"> <label class="md-overlay" data-md-component="overlay" for="__drawer"></label><a href="#categories" tabindex="1" class="md-skip">Skip to content </a><header class="md-header" data-md-component="header"><nav class="md-header-nav md-grid"><div class="md-flex"><div class="md-flex__cell md-flex__cell--shrink"><a href="https://www.sherlock.stanford.edu/" title="Sherlock" class="md-header-nav__button md-logo"><img src="../../../img/logo_small.png" width="24" height="24"></a></div><div class="md-flex__cell md-flex__cell--shrink"><label class="md-icon md-icon--menu md-header-nav__button" for="__drawer"></label></div><div class="md-flex__cell md-flex__cell--stretch"><div class="md-flex__ellipsis md-header-nav__title" data-md-component="title"><span class="md-header-nav__topic">Sherlock</span><span class="md-header-nav__topic"> list</span></div></div><div class="md-flex__cell md-flex__cell--shrink"><label class="md-icon md-icon--search md-header-nav__button" for="__search"></label><div class="md-search" data-md-component="search" role="dialog"><label class="md-search__overlay" for="__search"></label><div class="md-search__inner" role="search"><form class="md-search__form" name="search"><input type="text" class="md-search__input" name="query" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="query" data-md-state="active"> <label class="md-icon md-search__icon" for="__search"></label> <button type="reset" class="md-icon md-search__icon" data-md-component="reset" tabindex="-1">&#xE5CD;</button></form><div class="md-search__output"><div class="md-search__scrollwrap" data-md-scrollfix><div class="md-search-result" data-md-component="result"><div class="md-search-result__meta">Type to start searching</div><ol class="md-search-result__list"></ol></div></div></div></div></div></div><div class="md-flex__cell md-flex__cell--shrink"><div class="md-header-nav__source"><a href="https://github.com/stanford-rc/" title="Go to repository" class="md-source" data-md-source="github"><div class="md-source__icon"><svg viewBox="0 0 24 24" width="24" height="24"><use xlink:href="#__github" width="24" height="24"></use></svg></div><div class="md-source__repository">SRCC <i class="fa fa-heart-o"></i> Open Source</div></a></div></div></div></nav></header><div class="md-container"><main class="md-main"><div class="md-main__inner md-grid" data-md-component="container"><div class="md-sidebar md-sidebar--primary" data-md-component="navigation"><div class="md-sidebar__scrollwrap"><div class="md-sidebar__inner"><nav class="md-nav md-nav--primary" data-md-level="0"><label class="md-nav__title md-nav__title--site" for="__drawer"><a href="https://www.sherlock.stanford.edu/" title="Sherlock" class="md-nav__button md-logo"><img src="../../../img/logo_small.png" width="48" height="48"></a>Sherlock</label><div class="md-nav__source"><a href="https://github.com/stanford-rc/" title="Go to repository" class="md-source" data-md-source="github"><div class="md-source__icon"><svg viewBox="0 0 24 24" width="24" height="24"><use xlink:href="#__github" width="24" height="24"></use></svg></div><div class="md-source__repository">SRCC <i class="fa fa-heart-o"></i> Open Source</div></a></div><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-1" type="checkbox" id="nav-1"><label class="md-nav__link" for="nav-1">Overview</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-1">Overview</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../overview/introduction/" title="Introduction" class="md-nav__link">Introduction</a></li><li class="md-nav__item"><a href="../../overview/concepts/" title="Concepts" class="md-nav__link">Concepts</a></li><li class="md-nav__item"><a href="../../overview/glossary/" title="Glossary" class="md-nav__link">Glossary</a></li><li class="md-nav__item"><a href="../../overview/specs/" title="Tech specs" class="md-nav__link">Tech specs</a></li><li class="md-nav__item"><a href="../../overview/status/" title="Status" class="md-nav__link">Status</a></li><li class="md-nav__item"><a href="../../overview/about/" title="About" class="md-nav__link">About</a></li></ul></nav></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-2" type="checkbox" id="nav-2"><label class="md-nav__link" for="nav-2">Getting Started</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-2">Getting Started</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../getting-started/prerequisites/" title="Prerequisites" class="md-nav__link">Prerequisites</a></li><li class="md-nav__item"><a href="../../getting-started/connecting/" title="Connecting" class="md-nav__link">Connecting</a></li><li class="md-nav__item"><a href="../../getting-started/submitting/" title="Submitting jobs" class="md-nav__link">Submitting jobs</a></li></ul></nav></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-3" type="checkbox" id="nav-3"><label class="md-nav__link" for="nav-3">User guide</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-3">User guide</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../user-guide/running-jobs/" title="Running jobs" class="md-nav__link">Running jobs</a></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-3-2" type="checkbox" id="nav-3-2"><label class="md-nav__link" for="nav-3-2">Specialized resources</label><nav class="md-nav" data-md-component="collapsible" data-md-level="2"><label class="md-nav__title" for="nav-3-2">Specialized resources</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../user-guide/gpu/" title="GPU nodes" class="md-nav__link">GPU nodes</a></li></ul></nav></li><li class="md-nav__item"><a href="../../user-guide/ondemand/" title="OnDemand" class="md-nav__link">OnDemand</a></li><li class="md-nav__item"><a href="../../user-guide/troubleshoot/" title="Troubleshooting" class="md-nav__link">Troubleshooting</a></li></ul></nav></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-4" type="checkbox" id="nav-4"><label class="md-nav__link" for="nav-4">Storage</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-4">Storage</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../storage/overview/" title="Overview" class="md-nav__link">Overview</a></li><li class="md-nav__item"><a href="../../storage/filesystems/" title="Filesystems" class="md-nav__link">Filesystems</a></li><li class="md-nav__item"><a href="../../storage/data-sharing/" title="Data sharing" class="md-nav__link">Data sharing</a></li><li class="md-nav__item"><a href="../../storage/data-protection/" title="Data protection" class="md-nav__link">Data protection</a></li><li class="md-nav__item"><a href="../../storage/data-transfer/" title="Data transfer" class="md-nav__link">Data transfer</a></li></ul></nav></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-5" type="checkbox" id="nav-5"><label class="md-nav__link" for="nav-5">Software</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-5">Software</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../overview/" title="Overview" class="md-nav__link">Overview</a></li><li class="md-nav__item"><a href="../list/" title="List" class="md-nav__link">List</a></li><li class="md-nav__item"><a href="../modules/" title="Modules" class="md-nav__link">Modules</a></li><li class="md-nav__item"><a href="../install/" title="Installation" class="md-nav__link">Installation</a></li><li class="md-nav__item"><a href="../containers/" title="Containers" class="md-nav__link">Containers</a></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-5-6" type="checkbox" id="nav-5-6"><label class="md-nav__link" for="nav-5-6">Usage</label><nav class="md-nav" data-md-component="collapsible" data-md-level="2"><label class="md-nav__title" for="nav-5-6">Usage</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../using/R/" title="R" class="md-nav__link">R</a></li><li class="md-nav__item"><a href="../using/julia/" title="Julia" class="md-nav__link">Julia</a></li><li class="md-nav__item"><a href="../using/python/" title="Python" class="md-nav__link">Python</a></li><li class="md-nav__item"><a href="../using/perl/" title="Perl" class="md-nav__link">Perl</a></li><li class="md-nav__item"><a href="../using/spark/" title="Spark" class="md-nav__link">Spark</a></li><li class="md-nav__item"><a href="../using/mariadb/" title="MariaDB" class="md-nav__link">MariaDB</a></li><li class="md-nav__item"><a href="../using/postgresql/" title="PostgreSQL" class="md-nav__link">PostgreSQL</a></li><li class="md-nav__item"><a href="../using/singularity/" title="Singularity" class="md-nav__link">Singularity</a></li></ul></nav></li></ul></nav></li><li class="md-nav__item md-nav__item--nested"><input class="md-toggle md-nav__toggle" data-md-toggle="nav-6" type="checkbox" id="nav-6"><label class="md-nav__link" for="nav-6">Advanced topics</label><nav class="md-nav" data-md-component="collapsible" data-md-level="1"><label class="md-nav__title" for="nav-6">Advanced topics</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="../../advanced-topics/connection/" title="Connection" class="md-nav__link">Connection</a></li></ul></nav></li></ul></nav></div></div></div><div class="md-sidebar md-sidebar--secondary" data-md-component="toc"><div class="md-sidebar__scrollwrap"><div class="md-sidebar__inner"><nav class="md-nav md-nav--secondary"><label class="md-nav__title" for="__toc">Table of contents</label><ul class="md-nav__list" data-md-scrollfix><li class="md-nav__item"><a href="#categories" title="Categories" class="md-nav__link">Categories</a><nav class="md-nav"><ul class="md-nav__list"><li class="md-nav__item"><a href="#viz" title="Viz" class="md-nav__link">Viz</a></li><li class="md-nav__item"><a href="#system" title="System" class="md-nav__link">System</a></li><li class="md-nav__item"><a href="#math" title="Math" class="md-nav__link">Math</a></li><li class="md-nav__item"><a href="#physics" title="Physics" class="md-nav__link">Physics</a></li><li class="md-nav__item"><a href="#biology" title="Biology" class="md-nav__link">Biology</a></li><li class="md-nav__item"><a href="#chemistry" title="Chemistry" class="md-nav__link">Chemistry</a></li><li class="md-nav__item"><a href="#devel" title="Devel" class="md-nav__link">Devel</a></li></ul></nav></li></ul></nav></div></div></div><div class="md-content"><article class="md-content__inner md-typeset"><a href="https://github.com/stanford-rc/www.sherlock.stanford.edu/edit/docs/src/docs/software/_list.md" title="Edit this page" class="md-icon md-content__icon">&#xE3C9;</a><h1> list</h1><h2 id="categories">Categories<a class="headerlink" href="#categories" title="Permanent link">#</a></h2>
<p><em>As of Thursday, June 13 2019, we provide 529 software packages, in 7 categories, covering 61 fields of science:</em></p>
<ul>
<li><a href="#viz"><code>viz</code></a> <small>graphs, imaging, molecular visualization, plotting, remote display</small></li>
<li><a href="#system"><code>system</code></a> <small>benchmark, benchmarks, compression, containers, database, doc, file management, file transfer, framework, language, libs, media, performance, resource monitoring, scm, tools</small></li>
<li><a href="#math"><code>math</code></a> <small>computational geometry, deep learning, linear algebra, machine learning, numerical analysis, numerical library, optimization, scientific computing, statistics, symbolic</small></li>
<li><a href="#physics"><code>physics</code></a> <small>astronomy, geophysics, geoscience, materials science, photonics</small></li>
<li><a href="#biology"><code>biology</code></a> <small>clinical science, computational biology, cryo-em, genomics, neurology, pathology, phylogenetics</small></li>
<li><a href="#chemistry"><code>chemistry</code></a> <small>cheminformatics, computational chemistry, cristallography, electrostatics, molecular dynamics, quantum chemistry</small></li>
<li><a href="#devel"><code>devel</code></a> <small>build, compiler, data, data analytics, debug, framework, language, lib, mpi, networking, parser, sdk</small></li>
</ul>
<div class="admonition warning">
<p class="admonition-title">Licensed software</p>
<p>Access to software modules marked with <sup><b class="lic"></b></sup> in the tables
below is restricted to properly licensed user groups.</p>
<p>The SRCC is not funded to provide commercial software on Sherlock and
researchers are responsible for the costs of purchasing and renewing
commercial software licenses. For more information, please feel free to
<a href="mailto:srcc-support@stanford.edu">contact us</a> and see the <a href="https://uit.stanford.edu/service/softwarelic">Stanford
Software Licensing page</a> for
purchasing information.</p>
</div>
<!-- color styles for module properties -->

<style>
.lic  { color: darkred; }
.lic:after {
    content: attr(class);
}
.soft_anch {
    display: inline-block;
    padding-top: 70px;
    margin-top: -70px;
}
</style>

<h3 id="viz">Viz<a class="headerlink" href="#viz" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>graphs</strong></td>
<td align="left"><a id="graphviz" class="soft_anch"><code>graphviz</code></a></td>
<td align="left">-&nbsp;<code>2.40.1</code></td>
<td align="left"><a href="https://www.graphviz.org/">Website</a></td>
<td align="left">Graphviz is open source graph visualization software.</td>
</tr>
<tr>
<td align="left"><strong>imaging</strong></td>
<td align="left"><a id="py-pillow" class="soft_anch"><code>py-pillow</code></a></td>
<td align="left">-&nbsp;<code>5.1.0_py36</code><br/>-&nbsp;<code>5.1.0_py27</code></td>
<td align="left"><a href="https://python-pillow.org">Website</a></td>
<td align="left">Pillow is a friendly PIL (Python Imaging Library) fork.</td>
</tr>
<tr>
<td align="left"><strong>molecular visualization</strong></td>
<td align="left"><a id="pymol" class="soft_anch"><code>pymol</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>1.8.6.2</code></td>
<td align="left"><a href="https://pymol.org">Website</a></td>
<td align="left">PyMOL is a Python-enhanced molecular graphics tool.</td>
</tr>
<tr>
<td align="left"><strong>plotting</strong></td>
<td align="left"><a id="gnuplot" class="soft_anch"><code>gnuplot</code></a></td>
<td align="left">-&nbsp;<code>5.2.0</code></td>
<td align="left"><a href="http://www.gnuplot.info">Website</a></td>
<td align="left">Gnuplot is a portable command-line driven graphing utility for Linux, OS/2, MS Windows, OSX, VMS, and many other platforms.</td>
</tr>
<tr>
<td align="left"><strong>plotting</strong></td>
<td align="left"><a id="grace" class="soft_anch"><code>grace</code></a></td>
<td align="left">-&nbsp;<code>5.1.25</code></td>
<td align="left"><a href="http://plasma-gate.weizmann.ac.il/Grace">Website</a></td>
<td align="left">Grace is a WYSIWYG tool to make two-dimensional plots of numerical data.</td>
</tr>
<tr>
<td align="left"><strong>plotting</strong></td>
<td align="left"><a id="py-basemap" class="soft_anch"><code>py-basemap</code></a></td>
<td align="left">-&nbsp;<code>1.1.0_py36</code><br/>-&nbsp;<code>1.1.0_py27</code></td>
<td align="left"><a href="https://matplotlib.org/basemap/">Website</a></td>
<td align="left">The matplotlib basemap toolkit is a library for plotting 2D data on maps in Python.</td>
</tr>
<tr>
<td align="left"><strong>plotting</strong></td>
<td align="left"><a id="py-matplotlib" class="soft_anch"><code>py-matplotlib</code></a></td>
<td align="left">-&nbsp;<code>2.1.2_py27</code><br/>-&nbsp;<code>2.1.2_py36</code><br/>-&nbsp;<code>2.2.2_py36</code><br/>-&nbsp;<code>2.2.2_py27</code></td>
<td align="left"><a href="http://matplotlib.org">Website</a></td>
<td align="left">Matplotlib is a Python 2D plotting library which produces publication quality figures in a variety of hardcopy formats and interactive environments across platforms.</td>
</tr>
<tr>
<td align="left"><strong>plotting</strong></td>
<td align="left"><a id="py-plotly" class="soft_anch"><code>py-plotly</code></a></td>
<td align="left">-&nbsp;<code>2.4.1_py27</code></td>
<td align="left"><a href="https://plot.ly/python/">Website</a></td>
<td align="left">Plotly's Python graphing library makes interactive, publication-quality graphs online.</td>
</tr>
<tr>
<td align="left"><strong>remote display</strong></td>
<td align="left"><a id="virtualgl" class="soft_anch"><code>virtualgl</code></a></td>
<td align="left">-&nbsp;<code>2.5.2</code></td>
<td align="left"><a href="http://www.virtualgl.org">Website</a></td>
<td align="left">VirtualGL is an open source toolkit that gives any Unix or Linux remote display software the ability to run OpenGL applications with full 3D hardware acceleration.</td>
</tr>
</tbody>
</table>
<h3 id="system">System<a class="headerlink" href="#system" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>benchmark</strong></td>
<td align="left"><a id="hp2p" class="soft_anch"><code>hp2p</code></a></td>
<td align="left">-&nbsp;<code>3.2</code></td>
<td align="left"><a href="https://github.com/cea-hpc/hp2p">Website</a></td>
<td align="left">Heavy Peer To Peer: a MPI based benchmark for network diagnostic.</td>
</tr>
<tr>
<td align="left"><strong>benchmark</strong></td>
<td align="left"><a id="mprime" class="soft_anch"><code>mprime</code></a></td>
<td align="left">-&nbsp;<code>29.4</code></td>
<td align="left"><a href="https://www.mersenne.org/">Website</a></td>
<td align="left">mprime is used by GIMPS, a distributed computing project dedicated to finding new Mersenne prime numbers, and which is commonly used as a stability testing utility.</td>
</tr>
<tr>
<td align="left"><strong>benchmarks</strong></td>
<td align="left"><a id="osu-micro-benchmarks" class="soft_anch"><code>osu-micro-benchmarks</code></a></td>
<td align="left">-&nbsp;<code>5.6.1</code></td>
<td align="left"><a href="http://mvapich.cse.ohio-state.edu/benchmarks/">Website</a></td>
<td align="left">The OSU MicroBenchmarks carry out a variety of message passing performance tests using MPI.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="libarchive" class="soft_anch"><code>libarchive</code></a></td>
<td align="left">-&nbsp;<code>3.3.2</code></td>
<td align="left"><a href="https://www.libarchive.org/">Website</a></td>
<td align="left">The libarchive project develops a portable, efficient C library that can read and write streaming archives in a variety of formats.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="libzip" class="soft_anch"><code>libzip</code></a></td>
<td align="left">-&nbsp;<code>1.5.1</code></td>
<td align="left"><a href="http://libzip.org">Website</a></td>
<td align="left">libzip is a C library for reading, creating, and modifying zip archives.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="lz4" class="soft_anch"><code>lz4</code></a></td>
<td align="left">-&nbsp;<code>1.8.0</code></td>
<td align="left"><a href="https://www.lz4.org">Website</a></td>
<td align="left">LZ4 is lossless compression algorithm.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="lzo" class="soft_anch"><code>lzo</code></a></td>
<td align="left">-&nbsp;<code>2.10</code></td>
<td align="left"><a href="http://www.oberhumer.com/opensource/lzo">Website</a></td>
<td align="left">LZO is a portable lossless data compression library written in ANSI C.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="mpibzip2" class="soft_anch"><code>mpibzip2</code></a></td>
<td align="left">-&nbsp;<code>0.6</code></td>
<td align="left"><a href="https://compression.ca/mpibzip2">Website</a></td>
<td align="left">MPIBZIP2 is a parallel implementation of the bzip2 block-sorting file compressor that uses MPI and achieves significant speedup on cluster machines.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="p7zip" class="soft_anch"><code>p7zip</code></a></td>
<td align="left">-&nbsp;<code>16.02</code></td>
<td align="left"><a href="https://sourceforge.net/projects/p7zip/">Website</a></td>
<td align="left">p7zip is a Linux port of 7zip, a file archiver with high compression ratio.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="pbzip2" class="soft_anch"><code>pbzip2</code></a></td>
<td align="left">-&nbsp;<code>1.1.12</code></td>
<td align="left"><a href="https://compression.ca/pbzip2">Website</a></td>
<td align="left">PBZIP2 is a parallel implementation of the bzip2 block-sorting file compressor that uses pthreads and achieves near-linear speedup on SMP machines.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="pigz" class="soft_anch"><code>pigz</code></a></td>
<td align="left">-&nbsp;<code>2.4</code></td>
<td align="left"><a href="https://zlib.net/pigz/">Website</a></td>
<td align="left">A parallel implementation of gzip for modern multi-processor, multi-core machines.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="szip" class="soft_anch"><code>szip</code></a></td>
<td align="left">-&nbsp;<code>2.1.1</code></td>
<td align="left"><a href="https://support.hdfgroup.org/doc_resource/SZIP">Website</a></td>
<td align="left">Szip compression software, providing lossless compression of scientific data, is an implementation of the extended-Rice lossless compression algorithm.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="xz" class="soft_anch"><code>xz</code></a></td>
<td align="left">-&nbsp;<code>5.2.3</code></td>
<td align="left"><a href="http://tukaani.org/xz/">Website</a></td>
<td align="left">XZ Utils is free general-purpose data compression software with a high compression ratio.</td>
</tr>
<tr>
<td align="left"><strong>compression</strong></td>
<td align="left"><a id="zlib" class="soft_anch"><code>zlib</code></a></td>
<td align="left">-&nbsp;<code>1.2.11</code></td>
<td align="left"><a href="http://zlib.net">Website</a></td>
<td align="left">zlib is designed to be a free, general-purpose, legally unencumbered -- that is, not covered by any patents -- lossless data-compression library for use on virtually any computer hardware and operating system.</td>
</tr>
<tr>
<td align="left"><strong>containers</strong></td>
<td align="left"><a id="libnvidia-container" class="soft_anch"><code>libnvidia-container</code></a></td>
<td align="left">-&nbsp;<code>1.0.0rc2</code></td>
<td align="left"><a href="https://github.com/NVIDIA/libnvidia-container">Website</a></td>
<td align="left">libnvidia-container is a library and a simple CLI utility to automatically configure GNU/Linux containers leveraging NVIDIA hardware.</td>
</tr>
<tr>
<td align="left"><strong>containers</strong></td>
<td align="left"><a id="proot" class="soft_anch"><code>proot</code></a></td>
<td align="left">-&nbsp;<code>5.1.0</code></td>
<td align="left"><a href="https://proot-me.github.io">Website</a></td>
<td align="left">PRoot is a user-space implementation of chroot, mount --bind, and binfmt_misc.</td>
</tr>
<tr>
<td align="left"><strong>database</strong></td>
<td align="left"><a id="bdb" class="soft_anch"><code>bdb</code></a></td>
<td align="left">-&nbsp;<code>6.2.32</code></td>
<td align="left"><a href="http://www.oracle.com/us/products/database/berkeley-db">Website</a></td>
<td align="left">Berkeley DB (BDB) is a software library intended to provide a high-performance embedded database for key/value data.</td>
</tr>
<tr>
<td align="left"><strong>database</strong></td>
<td align="left"><a id="mariadb" class="soft_anch"><code>mariadb</code></a></td>
<td align="left">-&nbsp;<code>10.2.11</code></td>
<td align="left"><a href="https://www.mariadb.org">Website</a></td>
<td align="left">MariaDB is a community-developed fork of the MySQL relational database management system intended to remain free under the GNU GPL. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/mariadb">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>database</strong></td>
<td align="left"><a id="postgresql" class="soft_anch"><code>postgresql</code></a></td>
<td align="left">-&nbsp;<code>10.5</code></td>
<td align="left"><a href="https://postgresql.org">Website</a></td>
<td align="left">PostgreSQL is a powerful, open source object-relational database system with a strong focus on reliability, feature robustness, and performance. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/postgresql">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>database</strong></td>
<td align="left"><a id="sqlite" class="soft_anch"><code>sqlite</code></a></td>
<td align="left">-&nbsp;<code>3.18.0</code></td>
<td align="left"><a href="https://www.sqlite.org">Website</a></td>
<td align="left">SQLite is a self-contained, high-reliability, embedded, full-featured, public-domain, SQL database engine.</td>
</tr>
<tr>
<td align="left"><strong>doc</strong></td>
<td align="left"><a id="texinfo" class="soft_anch"><code>texinfo</code></a></td>
<td align="left">-&nbsp;<code>6.6</code></td>
<td align="left"><a href="https://www.gnu.org/software/texinfo/">Website</a></td>
<td align="left">Texinfo is the official documentation format of the GNU project.</td>
</tr>
<tr>
<td align="left"><strong>file management</strong></td>
<td align="left"><a id="duc" class="soft_anch"><code>duc</code></a></td>
<td align="left">-&nbsp;<code>1.4.4</code></td>
<td align="left"><a href="https://duc.zevv.nl">Website</a></td>
<td align="left">Duc is a collection of tools for indexing, inspecting and visualizing disk usage.</td>
</tr>
<tr>
<td align="left"><strong>file management</strong></td>
<td align="left"><a id="exa" class="soft_anch"><code>exa</code></a></td>
<td align="left">-&nbsp;<code>0.8.0</code></td>
<td align="left"><a href="https://github.com/ogham/exa">Website</a></td>
<td align="left">exa is a replacement for ls written in Rust.</td>
</tr>
<tr>
<td align="left"><strong>file management</strong></td>
<td align="left"><a id="fpart" class="soft_anch"><code>fpart</code></a></td>
<td align="left">-&nbsp;<code>0.9.3</code></td>
<td align="left"><a href="https://github.com/martymac/fpart">Website</a></td>
<td align="left">fpart sorts files and packs them into partitions.</td>
</tr>
<tr>
<td align="left"><strong>file management</strong></td>
<td align="left"><a id="py-pcircle" class="soft_anch"><code>py-pcircle</code></a></td>
<td align="left">-&nbsp;<code>0.17_py27</code></td>
<td align="left"><a href="https://github.com/olcf/pcircle">Website</a></td>
<td align="left">pcircle contains a suite of file system tools developed at OLCF to take advantage of highly scalable parallel file system such as Lustre.</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="gdrive" class="soft_anch"><code>gdrive</code></a></td>
<td align="left">-&nbsp;<code>2.1.0</code></td>
<td align="left"><a href="https://github.com/prasmussen/gdrive">Website</a></td>
<td align="left">gdrive is a command line utility for interacting with Google Drive.</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="gsutil" class="soft_anch"><code>gsutil</code></a></td>
<td align="left">-&nbsp;<code>4.31</code></td>
<td align="left"><a href="https://cloud.google.com/storage/docs/gsutil">Website</a></td>
<td align="left">gsutil is a Python application that lets you access Cloud Storage from the command line.</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="lftp" class="soft_anch"><code>lftp</code></a></td>
<td align="left">-&nbsp;<code>4.8.1</code></td>
<td align="left"><a href="https://lftp.yar.ru/">Website</a></td>
<td align="left">LFTP is a sophisticated file transfer program supporting a number of network protocols (ftp, http, sftp, fish, torrent).</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="mpifileutils" class="soft_anch"><code>mpifileutils</code></a></td>
<td align="left">-&nbsp;<code>20170210</code></td>
<td align="left"><a href="https://github.com/hpc/mpifileutils">Website</a></td>
<td align="left">mpiFileUtils is a suite of MPI-based tools to manage large datasets, which may vary from large directory trees to large files.</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="py-globus-cli" class="soft_anch"><code>py-globus-cli</code></a></td>
<td align="left">-&nbsp;<code>1.2.0</code><br/>-&nbsp;<code>1.9.0_py27</code><br/>-&nbsp;<code>1.9.0_py36</code></td>
<td align="left"><a href="https://github.com/globus/globus-cli">Website</a></td>
<td align="left">A command line wrapper over the Globus SDK for Python.</td>
</tr>
<tr>
<td align="left"><strong>file transfer</strong></td>
<td align="left"><a id="rclone" class="soft_anch"><code>rclone</code></a></td>
<td align="left">-&nbsp;<code>1.39</code><br/>-&nbsp;<code>1.43.1</code></td>
<td align="left"><a href="https://rclone.org">Website</a></td>
<td align="left">Rclone is a command line program to sync files and directories to and from: Google Drive, Amazon S3, Dropbox, Google Cloud Storage, Amazon Drive, Microsoft One Drive, Hubic, Backblaze B2, Yandex Disk, or the local filesystem.</td>
</tr>
<tr>
<td align="left"><strong>framework</strong></td>
<td align="left"><a id="mono" class="soft_anch"><code>mono</code></a></td>
<td align="left">-&nbsp;<code>5.12.0.301</code><br/>-&nbsp;<code>5.20.1.19</code></td>
<td align="left"><a href="https://www.mono-project.com">Website</a></td>
<td align="left">Mono is an open source implementation of Microsoft's .NET Framework based on the ECMA standards for C# and the Common Language Runtime.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="tcltk" class="soft_anch"><code>tcltk</code></a></td>
<td align="left">-&nbsp;<code>8.6.6</code></td>
<td align="left"><a href="https://www.tcl.tk">Website</a></td>
<td align="left">Tcl (Tool Command Language) is a dynamic programming language, suitable for web and desktop applications, networking, administration, testing. Tk is a graphical user interface toolkit.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="apr" class="soft_anch"><code>apr</code></a></td>
<td align="left">-&nbsp;<code>1.6.3</code></td>
<td align="left"><a href="https://apr.apache.org/">Website</a></td>
<td align="left">The Apache Portable Runtime is a supporting library for the Apache web server. It provides a set of APIs that map to the underlying operating system.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="apr-util" class="soft_anch"><code>apr-util</code></a></td>
<td align="left">-&nbsp;<code>1.6.1</code></td>
<td align="left"><a href="https://apr.apache.org/">Website</a></td>
<td align="left">The Apache Portable Runtime is a supporting library for the Apache web server. It provides a set of APIs that map to the underlying operating system.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="atk" class="soft_anch"><code>atk</code></a></td>
<td align="left">-&nbsp;<code>2.24.0</code></td>
<td align="left"><a href="https://developer.gnome.org/atk">Website</a></td>
<td align="left">ATK is the Accessibility Toolkit. It provides a set of generic interfaces allowing accessibility technologies such as screen readers to interact with a graphical user interface.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="benchmark" class="soft_anch"><code>benchmark</code></a></td>
<td align="left">-&nbsp;<code>1.2.0</code></td>
<td align="left"><a href="https://github.com/google/benchmark">Website</a></td>
<td align="left">A microbenchmark support library</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="cairo" class="soft_anch"><code>cairo</code></a></td>
<td align="left">-&nbsp;<code>1.14.10</code></td>
<td align="left"><a href="https://www.cairographics.org/">Website</a></td>
<td align="left">Cairo is a 2D graphics library with support for multiple output devices.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="cups" class="soft_anch"><code>cups</code></a></td>
<td align="left">-&nbsp;<code>2.2.4</code></td>
<td align="left"><a href="https://www.cups.org/">Website</a></td>
<td align="left">CUPS is the standards-based, open source printing system.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="dbus" class="soft_anch"><code>dbus</code></a></td>
<td align="left">-&nbsp;<code>1.10.22</code></td>
<td align="left"><a href="https://www.freedesktop.org/wiki/Software/dbus/">Website</a></td>
<td align="left">D-Bus is a message bus system, a simple way for applications to talk to one another.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="enchant" class="soft_anch"><code>enchant</code></a></td>
<td align="left">-&nbsp;<code>1.6.1</code><br/>-&nbsp;<code>2.2.3</code></td>
<td align="left"><a href="https://abiword.github.io/enchant/">Website</a></td>
<td align="left">Enchant is a library (and command-line program) that wraps a number of different spelling libraries and programs with a consistent interface.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="fltk" class="soft_anch"><code>fltk</code></a></td>
<td align="left">-&nbsp;<code>1.3.4</code></td>
<td align="left"><a href="http://www.fltk.org">Website</a></td>
<td align="left">FLTK (pronounced 'fulltick') is a cross-platform C++ GUI toolkit.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="fontconfig" class="soft_anch"><code>fontconfig</code></a></td>
<td align="left">-&nbsp;<code>2.12.4</code></td>
<td align="left"><a href="https://www.freedesktop.org/wiki/Software/fontconfig">Website</a></td>
<td align="left">Fontconfig is a library for configuring and customizing font access.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="freeglut" class="soft_anch"><code>freeglut</code></a></td>
<td align="left">-&nbsp;<code>3.0.0</code></td>
<td align="left"><a href="http://freeglut.sourceforge.net/">Website</a></td>
<td align="left">FreeGLUT is a free-software/open-source alternative to the OpenGL Utility Toolkit (GLUT) library.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="freetype" class="soft_anch"><code>freetype</code></a></td>
<td align="left">-&nbsp;<code>2.8</code><br/>-&nbsp;<code>2.9.1</code></td>
<td align="left"><a href="http://freetype.org">Website</a></td>
<td align="left">FreeType is a software font engine that is designed to be small, efficient, highly customizable, and portable while capable of producing high-quality output (glyph images).</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gc" class="soft_anch"><code>gc</code></a></td>
<td align="left">-&nbsp;<code>7.6.0</code></td>
<td align="left"><a href="http://www.hboehm.info/gc">Website</a></td>
<td align="left">The Boehm-Demers-Weiser conservative garbage collector can be used as a garbage collecting replacement for C malloc or C++ new.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gconf" class="soft_anch"><code>gconf</code></a></td>
<td align="left">-&nbsp;<code>2.9.91</code></td>
<td align="left"><a href="https://projects.gnome.org/gconf">Website</a></td>
<td align="left">GConf is a system for storing application preferences.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gdk-pixbuf" class="soft_anch"><code>gdk-pixbuf</code></a></td>
<td align="left">-&nbsp;<code>2.36.8</code></td>
<td align="left"><a href="https://git.gnome.org/browse/gdk-pixbuf">Website</a></td>
<td align="left">The GdkPixbuf library provides facilities for loading images in a variety of file formats.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gflags" class="soft_anch"><code>gflags</code></a></td>
<td align="left">-&nbsp;<code>2.2.1</code></td>
<td align="left"><a href="https://gflags.github.io/gflags/">Website</a></td>
<td align="left">The gflags package contains a C++ library that implements commandline flags processing.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="giflib" class="soft_anch"><code>giflib</code></a></td>
<td align="left">-&nbsp;<code>5.1.4</code></td>
<td align="left"><a href="http://giflib.sourceforge.net/">Website</a></td>
<td align="left">GIFLIB is a package of portable tools and library routines for working with GIF images.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="glib" class="soft_anch"><code>glib</code></a></td>
<td align="left">-&nbsp;<code>2.52.3</code></td>
<td align="left"><a href="https://wiki.gnome.org/Projects/GLib">Website</a></td>
<td align="left">The GLib library provides core non-graphical functionality such as high level data types, Unicode manipulation, and an object and type system to C programs.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="glog" class="soft_anch"><code>glog</code></a></td>
<td align="left">-&nbsp;<code>0.3.5</code></td>
<td align="left"><a href="https://github.com/google/glog">Website</a></td>
<td align="left">C++ implementation of the Google logging module.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gnutls" class="soft_anch"><code>gnutls</code></a></td>
<td align="left">-&nbsp;<code>3.5.9</code></td>
<td align="left"><a href="https://www.gnutls.org">Website</a></td>
<td align="left">GnuTLS is a secure communications library implementing the SSL, TLS and DTLS protocols and technologies around them.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gobject-introspection" class="soft_anch"><code>gobject-introspection</code></a></td>
<td align="left">-&nbsp;<code>1.52.1</code></td>
<td align="left"><a href="https://wiki.gnome.org/Projects/GObjectIntrospection">Website</a></td>
<td align="left">GObject introspection is a middleware layer between C libraries (using GObject) and language bindings.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="googletest" class="soft_anch"><code>googletest</code></a></td>
<td align="left">-&nbsp;<code>1.8.0</code></td>
<td align="left"><a href="https://github.com/google/googletest">Website</a></td>
<td align="left">Google Test is Google's C++ test framework.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="gtk+" class="soft_anch"><code>gtk+</code></a></td>
<td align="left">-&nbsp;<code>2.24.30</code><br/>-&nbsp;<code>3.22.18</code></td>
<td align="left"><a href="https://www.gtk.org">Website</a></td>
<td align="left">GTK+, or the GIMP Toolkit, is a multi-platform toolkit for creating graphical user interfaces.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="harfbuzz" class="soft_anch"><code>harfbuzz</code></a></td>
<td align="left">-&nbsp;<code>1.4.8</code></td>
<td align="left"><a href="https://www.freedesktop.org/wiki/Software/HarfBuzz">Website</a></td>
<td align="left">HarfBuzz is an OpenType text shaping engine.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="hunspell" class="soft_anch"><code>hunspell</code></a></td>
<td align="left">-&nbsp;<code>1.6.2</code></td>
<td align="left"><a href="https://hunspell.github.io/">Website</a></td>
<td align="left">Hunspell is a spell checker.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="hyphen" class="soft_anch"><code>hyphen</code></a></td>
<td align="left">-&nbsp;<code>2.8.8</code></td>
<td align="left"><a href="https://github.com/hunspell/hyphen">Website</a></td>
<td align="left">Hyphen is a hyphenation library to use converted TeX hyphenation patterns.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="icu" class="soft_anch"><code>icu</code></a></td>
<td align="left">-&nbsp;<code>59.1</code></td>
<td align="left"><a href="http://site.icu-project.org">Website</a></td>
<td align="left">ICU is a set of C/C++ and Java libraries providing Unicode and Globalization support for software applications.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libaio" class="soft_anch"><code>libaio</code></a></td>
<td align="left">-&nbsp;<code>0.3.111</code></td>
<td align="left"><a href="https://pagure.io/libaio">Website</a></td>
<td align="left">libaio provides the Linux-native API for async I/O.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libepoxy" class="soft_anch"><code>libepoxy</code></a></td>
<td align="left">-&nbsp;<code>1.4.1</code></td>
<td align="left"><a href="https://github.com/anholt/libepoxy">Website</a></td>
<td align="left">Epoxy is a library for handling OpenGL function pointer management for you.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libexif" class="soft_anch"><code>libexif</code></a></td>
<td align="left">-&nbsp;<code>0.6.21</code></td>
<td align="left"><a href="https://github.com/libexif/libexif">Website</a></td>
<td align="left">A library for parsing, editing, and saving EXIF data.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libffi" class="soft_anch"><code>libffi</code></a></td>
<td align="left">-&nbsp;<code>3.2.1</code></td>
<td align="left"><a href="http://sourceware.org/libffi">Website</a></td>
<td align="left">libffi is a portable Foreign Function Interface library.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libgcrypt" class="soft_anch"><code>libgcrypt</code></a></td>
<td align="left">-&nbsp;<code>1.8.2</code></td>
<td align="left"><a href="https://www.gnupg.org/software/libgcrypt">Website</a></td>
<td align="left">Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libgd" class="soft_anch"><code>libgd</code></a></td>
<td align="left">-&nbsp;<code>2.2.5</code></td>
<td align="left"><a href="https://libgd.github.io/">Website</a></td>
<td align="left">GD is an open source code library for the dynamic creation of images by programmers.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libgdiplus" class="soft_anch"><code>libgdiplus</code></a></td>
<td align="left">-&nbsp;<code>5.6</code></td>
<td align="left"><a href="https://github.com/mono/libgdiplus">Website</a></td>
<td align="left">C-based implementation of the GDI+ API</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libgpg-error" class="soft_anch"><code>libgpg-error</code></a></td>
<td align="left">-&nbsp;<code>1.27</code></td>
<td align="left"><a href="https://www.gnupg.org/software/libgpg-error">Website</a></td>
<td align="left">Libgpg-error is a small library that originally defined common error values for all GnuPG components.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libidl" class="soft_anch"><code>libidl</code></a></td>
<td align="left">-&nbsp;<code>0.8.14</code></td>
<td align="left"><a href="https://github.com/GNOME/libIDL">Website</a></td>
<td align="left">The libIDL package contains libraries for Interface Definition Language files. This is a specification for defining portable interfaces.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libjpeg-turbo" class="soft_anch"><code>libjpeg-turbo</code></a></td>
<td align="left">-&nbsp;<code>1.5.1</code></td>
<td align="left"><a href="http://www.libjpeg-turbo.org">Website</a></td>
<td align="left">libjpeg-turbo is a JPEG image codec that uses SIMD instructions (MMX, SSE2, AVX2, NEON, AltiVec) to accelerate baseline JPEG compression and decompression on x86, x86-64, ARM, and PowerPC systems</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libmng" class="soft_anch"><code>libmng</code></a></td>
<td align="left">-&nbsp;<code>2.0.3</code></td>
<td align="left"><a href="http://libmng.sourceforge.net">Website</a></td>
<td align="left">THE reference library for reading, displaying, writing and examining Multiple-Image Network Graphics. MNG is the animation extension to the popular PNG image-format.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libpng" class="soft_anch"><code>libpng</code></a></td>
<td align="left">-&nbsp;<code>1.2.57</code><br/>-&nbsp;<code>1.6.29</code></td>
<td align="left"><a href="http://libpng.sourceforge.net">Website</a></td>
<td align="left">libpng is the official PNG reference library. It supports almost all PNG features, is extensible, and has been extensively tested for over 20 years.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libproxy" class="soft_anch"><code>libproxy</code></a></td>
<td align="left">-&nbsp;<code>0.4.15</code></td>
<td align="left"><a href="https://libproxy.github.io/libproxy/">Website</a></td>
<td align="left">libproxy is a library that provides automatic proxy configuration management.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libressl" class="soft_anch"><code>libressl</code></a></td>
<td align="left">-&nbsp;<code>2.5.3</code></td>
<td align="left"><a href="https://www.libressl.org">Website</a></td>
<td align="left">LibreSSL is a version of the TLS/crypto stack forked from OpenSSL in 2014, with goals of modernizing the codebase, improving security, and applying best practice development processes.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libseccomp" class="soft_anch"><code>libseccomp</code></a></td>
<td align="left">-&nbsp;<code>2.3.3</code></td>
<td align="left"><a href="https://github.com/seccomp/libseccomp">Website</a></td>
<td align="left">The libseccomp library provides an easy to use, platform independent, interface to the Linux Kernel's syscall filtering mechanism..</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libsoup" class="soft_anch"><code>libsoup</code></a></td>
<td align="left">-&nbsp;<code>2.61.2</code></td>
<td align="left"><a href="https://wiki.gnome.org/LibSoup">Website</a></td>
<td align="left">libsoup is an HTTP client/server library for GNOME.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libtasn1" class="soft_anch"><code>libtasn1</code></a></td>
<td align="left">-&nbsp;<code>4.13</code></td>
<td align="left"><a href="https://www.gnu.org/software/libtasn1/">Website</a></td>
<td align="left">Libtasn1 is the ASN.1 library used by GnuTLS, p11-kit and some other packages.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libtiff" class="soft_anch"><code>libtiff</code></a></td>
<td align="left">-&nbsp;<code>4.0.8</code></td>
<td align="left"><a href="http://simplesystems.org/libtiff/">Website</a></td>
<td align="left">libtiff provides support for the Tag Image File Format (TIFF), a widely used format for storing image data.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libunistring" class="soft_anch"><code>libunistring</code></a></td>
<td align="left">-&nbsp;<code>0.9.7</code></td>
<td align="left"><a href="https://www.gnu.org/software/libunistring/">Website</a></td>
<td align="left">Libunistring provides functions for manipulating Unicode strings and for manipulating C strings according to the Unicode standard.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libuuid" class="soft_anch"><code>libuuid</code></a></td>
<td align="left">-&nbsp;<code>1.0.3</code></td>
<td align="left"><a href="https://sourceforge.net/projects/libuuid">Website</a></td>
<td align="left">Portable uuid C library.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libwebp" class="soft_anch"><code>libwebp</code></a></td>
<td align="left">-&nbsp;<code>0.6.1</code></td>
<td align="left"><a href="https://developers.google.com/speed/webp">Website</a></td>
<td align="left">WebP is a modern image format that provides superior lossless and lossy compression for images on the web.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libxml2" class="soft_anch"><code>libxml2</code></a></td>
<td align="left">-&nbsp;<code>2.9.4</code></td>
<td align="left"><a href="http://xmlsoft.org/">Website</a></td>
<td align="left">Libxml2 is a XML C parser and toolkit.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="libxslt" class="soft_anch"><code>libxslt</code></a></td>
<td align="left">-&nbsp;<code>1.1.32</code></td>
<td align="left"><a href="http://xmlsoft.org/libxslt">Website</a></td>
<td align="left">Libxslt is the XSLT C library developed for the GNOME project. XSLT itself is a an XML language to define transformation for XML.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="mesa" class="soft_anch"><code>mesa</code></a></td>
<td align="left">-&nbsp;<code>17.1.6</code></td>
<td align="left"><a href="https://www.mesa3d.org">Website</a></td>
<td align="left">Mesa is an open-source implementation of the OpenGL, Vulkan and other specifications.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="ncurses" class="soft_anch"><code>ncurses</code></a></td>
<td align="left">-&nbsp;<code>6.0</code></td>
<td align="left"><a href="https://www.gnu.org/software/ncurses">Website</a></td>
<td align="left">The ncurses (new curses) library is a free software emulation of curses in System V Release 4.0 (SVr4), and more.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="nettle" class="soft_anch"><code>nettle</code></a></td>
<td align="left">-&nbsp;<code>3.3</code></td>
<td align="left"><a href="http://www.lysator.liu.se/~nisse/nettle/">Website</a></td>
<td align="left">Nettle is a cryptographic library that is designed to fit easily in more or less any context.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="openjpeg" class="soft_anch"><code>openjpeg</code></a></td>
<td align="left">-&nbsp;<code>2.3.1</code></td>
<td align="left"><a href="https://github.com/uclouvain/openjpeg">Website</a></td>
<td align="left">OpenJPEG is an open-source JPEG 2000 codec written in C language.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="orbit" class="soft_anch"><code>orbit</code></a></td>
<td align="left">-&nbsp;<code>2.14.19</code></td>
<td align="left"><a href="https://projects.gnome.org/ORBit2">Website</a></td>
<td align="left">ORBit2 is a CORBA 2.4-compliant Object Request Broker (ORB) featuring mature C, C++ and Python bindings.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="pango" class="soft_anch"><code>pango</code></a></td>
<td align="left">-&nbsp;<code>1.40.10</code></td>
<td align="left"><a href="http://www.pango.org">Website</a></td>
<td align="left">Pango is a library for laying out and rendering of text, with an emphasis on internationalization.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="pcre" class="soft_anch"><code>pcre</code></a></td>
<td align="left">-&nbsp;<code>8.40</code></td>
<td align="left"><a href="http://www.pcre.org/">Website</a></td>
<td align="left">The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax and semantics as Perl 5.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="popt" class="soft_anch"><code>popt</code></a></td>
<td align="left">-&nbsp;<code>1.16</code></td>
<td align="left"><a href="http://rpm5.org/files/popt">Website</a></td>
<td align="left">Library for parsing command line options.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="py-lmdb" class="soft_anch"><code>py-lmdb</code></a></td>
<td align="left">-&nbsp;<code>0.93</code></td>
<td align="left"><a href="https://lmdb.readthedocs.io/en/release/">Website</a></td>
<td align="left">Universal Python binding for the LMDB 'Lightning' Database.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="py-mako" class="soft_anch"><code>py-mako</code></a></td>
<td align="left">-&nbsp;<code>1.0.7_py36</code><br/>-&nbsp;<code>1.0.7_py27</code></td>
<td align="left"><a href="http://www.makotemplates.org/">Website</a></td>
<td align="left">Mako is a template library written in Python. It provides a familiar, non-XML syntax which compiles into Python modules for maximum performance.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="py-pyqt5" class="soft_anch"><code>py-pyqt5</code></a></td>
<td align="left">-&nbsp;<code>5.9.1_py36</code></td>
<td align="left"><a href="https://www.riverbankcomputing.com/software/pyqt/">Website</a></td>
<td align="left">PyQt5 is a comprehensive set of Python bindings for Qt v5.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="readline" class="soft_anch"><code>readline</code></a></td>
<td align="left">-&nbsp;<code>7.0</code></td>
<td align="left"><a href="https://cnswww.cns.cwru.edu/php/chet/readline/rltop.html">Website</a></td>
<td align="left">The GNU Readline library provides a set of functions for use by applications that allow users to edit command lines as they are typed in.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="snappy" class="soft_anch"><code>snappy</code></a></td>
<td align="left">-&nbsp;<code>1.1.7</code></td>
<td align="left"><a href="https://github.com/google/snappy">Website</a></td>
<td align="left">A fast compressor/decompressor.</td>
</tr>
<tr>
<td align="left"><strong>libs</strong></td>
<td align="left"><a id="talloc" class="soft_anch"><code>talloc</code></a></td>
<td align="left">-&nbsp;<code>2.1.14</code></td>
<td align="left"><a href="https://talloc.samba.org">Website</a></td>
<td align="left">talloc is a hierarchical, reference counted memory pool system with destructors.</td>
</tr>
<tr>
<td align="left"><strong>media</strong></td>
<td align="left"><a id="ffmpeg" class="soft_anch"><code>ffmpeg</code></a></td>
<td align="left">-&nbsp;<code>4.0</code></td>
<td align="left"><a href="https://www.ffmpeg.org">Website</a></td>
<td align="left">FFmpeg is the leading multimedia framework, able to decode, encode, transcode, mux, demux, stream, filter and play pretty much anything that humans and machines have created.</td>
</tr>
<tr>
<td align="left"><strong>media</strong></td>
<td align="left"><a id="libsndfile" class="soft_anch"><code>libsndfile</code></a></td>
<td align="left">-&nbsp;<code>1.0.28</code></td>
<td align="left"><a href="http://www.mega-nerd.com/libsndfile">Website</a></td>
<td align="left">Libsndfile is a C library for reading and writing files containing sampled sound (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface.</td>
</tr>
<tr>
<td align="left"><strong>performance</strong></td>
<td align="left"><a id="likwid" class="soft_anch"><code>likwid</code></a></td>
<td align="left">-&nbsp;<code>4.3.2</code></td>
<td align="left"><a href="https://github.com/RRZE-HPC/likwid">Website</a></td>
<td align="left">Likwid is a simple toolsuite of command line applications for performance oriented programmers.</td>
</tr>
<tr>
<td align="left"><strong>resource monitoring</strong></td>
<td align="left"><a id="nvtop" class="soft_anch"><code>nvtop</code></a></td>
<td align="left">-&nbsp;<code>0.2.2</code><br/>-&nbsp;<code>1.0.0</code></td>
<td align="left"><a href="https://github.com/Syllo/nvtop">Website</a></td>
<td align="left">Nvtop stands for NVidia TOP, a (h)top like task monitor for NVIDIA GPUs.</td>
</tr>
<tr>
<td align="left"><strong>resource monitoring</strong></td>
<td align="left"><a id="remora" class="soft_anch"><code>remora</code></a></td>
<td align="left">-&nbsp;<code>1.8.2</code></td>
<td align="left"><a href="https://github.com/TACC/remora">Website</a></td>
<td align="left">Remora is a tool to monitor runtime resource utilization.</td>
</tr>
<tr>
<td align="left"><strong>scm</strong></td>
<td align="left"><a id="git" class="soft_anch"><code>git</code></a></td>
<td align="left">-&nbsp;<code>2.12.2</code></td>
<td align="left"><a href="http://git-scm.com">Website</a></td>
<td align="left">Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.</td>
</tr>
<tr>
<td align="left"><strong>scm</strong></td>
<td align="left"><a id="git-lfs" class="soft_anch"><code>git-lfs</code></a></td>
<td align="left">-&nbsp;<code>2.4.0</code></td>
<td align="left"><a href="https://git-lfs.github.com/">Website</a></td>
<td align="left">Git Large File Storage (LFS) replaces large files such as audio samples, videos, datasets, and graphics with text pointers inside Git, while storing the file contents on a remote server.</td>
</tr>
<tr>
<td align="left"><strong>scm</strong></td>
<td align="left"><a id="mercurial" class="soft_anch"><code>mercurial</code></a></td>
<td align="left">-&nbsp;<code>4.5.3</code></td>
<td align="left"><a href="https://www.mercurial-scm.org/">Website</a></td>
<td align="left">Mercurial is a free, distributed source control management tool.</td>
</tr>
<tr>
<td align="left"><strong>scm</strong></td>
<td align="left"><a id="subversion" class="soft_anch"><code>subversion</code></a></td>
<td align="left">-&nbsp;<code>1.9.7</code></td>
<td align="left"><a href="https://subversion.apache.org/">Website</a></td>
<td align="left">Subversion is an open source version control system.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="curl" class="soft_anch"><code>curl</code></a></td>
<td align="left">-&nbsp;<code>7.54.0</code></td>
<td align="left"><a href="https://curl.haxx.se/">Website</a></td>
<td align="left">curl is an open source command line tool and library for transferring data with URL syntax.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="expat" class="soft_anch"><code>expat</code></a></td>
<td align="left">-&nbsp;<code>2.2.3</code></td>
<td align="left"><a href="http://libexpat.github.io">Website</a></td>
<td align="left">Expat is a stream-oriented XML parser library written in C.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="graphicsmagick" class="soft_anch"><code>graphicsmagick</code></a></td>
<td align="left">-&nbsp;<code>1.3.26</code></td>
<td align="left"><a href="http://www.graphicsmagick.org/">Website</a></td>
<td align="left">GraphicsMagick is the swiss army knife of image processing.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="imagemagick" class="soft_anch"><code>imagemagick</code></a></td>
<td align="left">-&nbsp;<code>7.0.7-2</code></td>
<td align="left"><a href="https://www.imagemagick.org">Website</a></td>
<td align="left">ImageMagick is a free and open-source software suite for displaying, converting, and editing raster image and vector image files.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="leveldb" class="soft_anch"><code>leveldb</code></a></td>
<td align="left">-&nbsp;<code>1.20</code></td>
<td align="left"><a href="https://symas.com/lightning-memory-mapped-database/">Website</a></td>
<td align="left">Symas LMDB is an extraordinarily fast, memory-efficient database we developed for the Symas OpenLDAP Project.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="lmdb" class="soft_anch"><code>lmdb</code></a></td>
<td align="left">-&nbsp;<code>0.9.21</code></td>
<td align="left"><a href="https://symas.com/lightning-memory-mapped-database/">Website</a></td>
<td align="left">Symas LMDB is an extraordinarily fast, memory-efficient database we developed for the Symas OpenLDAP Project.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="motif" class="soft_anch"><code>motif</code></a></td>
<td align="left">-&nbsp;<code>2.3.7</code></td>
<td align="left"><a href="https://motif.ics.com">Website</a></td>
<td align="left">Motif is the toolkit for the Common Desktop Environment.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="parallel" class="soft_anch"><code>parallel</code></a></td>
<td align="left">-&nbsp;<code>20180122</code></td>
<td align="left"><a href="https://www.gnu.org/software/parallel/">Website</a></td>
<td align="left">GNU parallel is a shell tool for executing jobs in parallel using one or more computers.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="qt" class="soft_anch"><code>qt</code></a></td>
<td align="left">-&nbsp;<code>5.9.1</code></td>
<td align="left"><a href="https://www.qt.io">Website</a></td>
<td align="left">QT is a cross-platform application framework that is used for developing application software that can be run on various software and hardware platforms.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="ripgrep" class="soft_anch"><code>ripgrep</code></a></td>
<td align="left">-&nbsp;<code>11.0.1</code></td>
<td align="left"><a href="https://github.com/BurntSushi/ripgrep">Website</a></td>
<td align="left">ripgrep recursively searches directories for a regex pattern.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="rocksdb" class="soft_anch"><code>rocksdb</code></a></td>
<td align="left">-&nbsp;<code>5.7.3</code></td>
<td align="left"><a href="https://rocksdb.org">Website</a></td>
<td align="left">A library that provides an embeddable, persistent key-value store for fast storage.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="x11" class="soft_anch"><code>x11</code></a></td>
<td align="left">-&nbsp;<code>7.7</code></td>
<td align="left"><a href="https://www.x.org">Website</a></td>
<td align="left">The X.Org project provides an open source implementation of the X Window System.</td>
</tr>
<tr>
<td align="left"><strong>tools</strong></td>
<td align="left"><a id="xkeyboard-config" class="soft_anch"><code>xkeyboard-config</code></a></td>
<td align="left">-&nbsp;<code>2.21</code></td>
<td align="left"><a href="https://www.freedesktop.org/wiki/Software/XKeyboardConfig/">Website</a></td>
<td align="left">The non-arch keyboard configuration database for X Window.</td>
</tr>
</tbody>
</table>
<h3 id="math">Math<a class="headerlink" href="#math" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>computational geometry</strong></td>
<td align="left"><a id="cgal" class="soft_anch"><code>cgal</code></a></td>
<td align="left">-&nbsp;<code>4.10</code></td>
<td align="left"><a href="http://www.cgal.org">Website</a></td>
<td align="left">The Computational Geometry Algorithms Library (CGAL) is a C++ library that aims to provide easy access to efficient and reliable algorithms in computational geometry.</td>
</tr>
<tr>
<td align="left"><strong>computational geometry</strong></td>
<td align="left"><a id="qhull" class="soft_anch"><code>qhull</code></a></td>
<td align="left">-&nbsp;<code>2015.2</code></td>
<td align="left"><a href="http://www.qhull.org/">Website</a></td>
<td align="left">Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point, furthest-site Delaunay triangulation, and furthest-site Voronoi diagram.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="caffe2" class="soft_anch"><code>caffe2</code></a></td>
<td align="left">-&nbsp;<code>0.8.1</code></td>
<td align="left"><a href="https://caffe2.ai">Website</a></td>
<td align="left">Caffe2 is a deep learning framework that provides an easy and straightforward way to experiment with deep learning and leverage community contributions of new models and algorithms.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="cudnn" class="soft_anch"><code>cudnn</code></a></td>
<td align="left">-&nbsp;<code>5.1</code><br/>-&nbsp;<code>6.0</code><br/>-&nbsp;<code>7.0.1</code><br/>-&nbsp;<code>7.0.4</code><br/>-&nbsp;<code>7.0.5</code><br/>-&nbsp;<code>7.1.1</code><br/>-&nbsp;<code>7.1.4</code><br/>-&nbsp;<code>7.3.1.20</code><br/>-&nbsp;<code>7.4.1.5</code></td>
<td align="left"><a href="https://developer.nvidia.com/cudnn">Website</a></td>
<td align="left">NVIDIA cuDNN is a GPU-accelerated library of primitives for deep neural networks.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="h2o" class="soft_anch"><code>h2o</code></a></td>
<td align="left">-&nbsp;<code>3.22.0.1</code></td>
<td align="left"><a href="https://www.tensorflow.org">Website</a></td>
<td align="left">TensorFlow is an open source software library for numerical computation using data flow graphs.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-horovod" class="soft_anch"><code>py-horovod</code></a></td>
<td align="left">-&nbsp;<code>0.12.1_py36</code><br/>-&nbsp;<code>0.12.1_py27</code></td>
<td align="left"><a href="https://github.com/uber/horovod">Website</a></td>
<td align="left">Horovod is a distributed training framework for TensorFlow. The goal of Horovod is to make distributed Deep Learning fast and easy to use.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-keras" class="soft_anch"><code>py-keras</code></a></td>
<td align="left">-&nbsp;<code>2.0.8_py27</code><br/>-&nbsp;<code>2.1.5_py36</code><br/>-&nbsp;<code>2.2.4_py27</code><br/>-&nbsp;<code>2.2.4_py36</code><br/>-&nbsp;<code>2.1.5_py27</code></td>
<td align="left"><a href="http://keras.io">Website</a></td>
<td align="left">Keras is a high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-onnx" class="soft_anch"><code>py-onnx</code></a></td>
<td align="left">-&nbsp;<code>1.0.1_py27</code></td>
<td align="left"><a href="https://onnx.ai">Website</a></td>
<td align="left">ONNX is a open format to represent deep learning models.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-pytorch" class="soft_anch"><code>py-pytorch</code></a></td>
<td align="left">-&nbsp;<code>0.2.0_py27</code><br/>-&nbsp;<code>0.2.0_py36</code><br/>-&nbsp;<code>0.3.0_py36</code><br/>-&nbsp;<code>1.0.0_py27</code><br/>-&nbsp;<code>1.0.0_py36</code><br/>-&nbsp;<code>0.3.0_py27</code></td>
<td align="left"><a href="http://pytorch.org">Website</a></td>
<td align="left">PyTorch is a deep learning framework that puts Python first.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-tensorflow" class="soft_anch"><code>py-tensorflow</code></a></td>
<td align="left">-&nbsp;<code>1.4.0_py27</code><br/>-&nbsp;<code>1.5.0_py27</code><br/>-&nbsp;<code>1.5.0_py36</code><br/>-&nbsp;<code>1.6.0_py36</code><br/>-&nbsp;<code>1.7.0_py27</code><br/>-&nbsp;<code>1.8.0_py27</code><br/>-&nbsp;<code>1.9.0_py27</code><br/>-&nbsp;<code>1.9.0_py36</code><br/>-&nbsp;<code>1.12.0_py27</code><br/>-&nbsp;<code>1.12.0_py36</code><br/>-&nbsp;<code>1.6.0_py27</code></td>
<td align="left"><a href="https://www.tensorflow.org">Website</a></td>
<td align="left">TensorFlow is an open source software library for numerical computation using data flow graphs.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-tensorlayer" class="soft_anch"><code>py-tensorlayer</code></a></td>
<td align="left">-&nbsp;<code>1.6.3_py27</code></td>
<td align="left"><a href="http://tensorlayer.org">Website</a></td>
<td align="left">TensorLayer is a Deep Learning (DL) and Reinforcement Learning (RL) library extended from Google TensorFlow.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="py-theano" class="soft_anch"><code>py-theano</code></a></td>
<td align="left">-&nbsp;<code>1.0.1_py27</code></td>
<td align="left"><a href="http://deeplearning.net/software/theano">Website</a></td>
<td align="left">Theano is a Python library that allows you to define, optimize, and evaluate mathematical expressions involving multi-dimensional arrays efficiently.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="tensorrt" class="soft_anch"><code>tensorrt</code></a></td>
<td align="left">-&nbsp;<code>3.0.1</code><br/>-&nbsp;<code>3.0.4</code><br/>-&nbsp;<code>4.0.1.6</code><br/>-&nbsp;<code>5.0.2.6</code></td>
<td align="left"><a href="https://developer.nvidia.com/tensorrt">Website</a></td>
<td align="left">NVIDIA TensorRT is a high-performance deep learning inference optimizer and runtime that delivers low latency, high-throughput inference for deep learning applications.</td>
</tr>
<tr>
<td align="left"><strong>deep learning</strong></td>
<td align="left"><a id="torch" class="soft_anch"><code>torch</code></a></td>
<td align="left">-&nbsp;<code>20180202</code></td>
<td align="left"><a href="http://torch.ch">Website</a></td>
<td align="left">Torch is a scientific computing framework with wide support for machine learning algorithms that puts GPUs first.</td>
</tr>
<tr>
<td align="left"><strong>linear algebra</strong></td>
<td align="left"><a id="armadillo" class="soft_anch"><code>armadillo</code></a></td>
<td align="left">-&nbsp;<code>8.200.1</code></td>
<td align="left"><a href="http://arma.sourceforge.net/">Website</a></td>
<td align="left">Armadillo is a high quality linear algebra library (matrix maths) for the C++ language, aiming towards a good balance between speed and ease of use.</td>
</tr>
<tr>
<td align="left"><strong>machine learning</strong></td>
<td align="left"><a id="py-scikit-learn" class="soft_anch"><code>py-scikit-learn</code></a></td>
<td align="left">-&nbsp;<code>0.19.1_py36</code><br/>-&nbsp;<code>0.19.1_py27</code></td>
<td align="left"><a href="http://scikit-learn.org">Website</a></td>
<td align="left">Scikit-learn is a free software machine learning library for the Python programming language.</td>
</tr>
<tr>
<td align="left"><strong>numerical analysis</strong></td>
<td align="left"><a id="matlab" class="soft_anch"><code>matlab</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>R2017a</code><br/>-&nbsp;<code>R2017b</code><br/>-&nbsp;<code>R2018a</code><br/>-&nbsp;<code>R2019a</code></td>
<td align="left"><a href="https://www.mathworks.com/products/matlab.html">Website</a></td>
<td align="left">MATLAB is a multi-paradigm numerical computing environment and proprietary programming language developed by MathWorks.</td>
</tr>
<tr>
<td align="left"><strong>numerical analysis</strong></td>
<td align="left"><a id="octave" class="soft_anch"><code>octave</code></a></td>
<td align="left">-&nbsp;<code>4.2.1</code></td>
<td align="left"><a href="https://www.gnu.org/software/octave/">Website</a></td>
<td align="left">GNU Octave is a high-level language primarily intended for numerical computations.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="arpack" class="soft_anch"><code>arpack</code></a></td>
<td align="left">-&nbsp;<code>3.5.0</code></td>
<td align="left"><a href="https://github.com/opencollab/arpack-ng">Website</a></td>
<td align="left">Collection of Fortran77 subroutines designed to solve large scale eigenvalue problems.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="fftw" class="soft_anch"><code>fftw</code></a></td>
<td align="left">-&nbsp;<code>3.3.6</code></td>
<td align="left"><a href="http://www.fftw.org/">Website</a></td>
<td align="left">The Fastest Fourier Transform in the West (FFTW) is a software library for computing discrete Fourier transforms (DFTs).</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="glpk" class="soft_anch"><code>glpk</code></a></td>
<td align="left">-&nbsp;<code>4.63</code></td>
<td align="left"><a href="http://www.openblas.net/">Website</a></td>
<td align="left">The GLPK (GNU Linear Programming Kit) package is intended for solving large-scale linear programming (LP), mixed integer programming (MIP), and other related problems.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="gmp" class="soft_anch"><code>gmp</code></a></td>
<td align="left">-&nbsp;<code>6.1.2</code></td>
<td align="left"><a href="https://gmplib.org">Website</a></td>
<td align="left">GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating-point numbers.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="gsl" class="soft_anch"><code>gsl</code></a></td>
<td align="left">-&nbsp;<code>1.16</code><br/>-&nbsp;<code>2.3</code></td>
<td align="left"><a href="https://www.gnu.org/software/gsl">Website</a></td>
<td align="left">The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers. The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="harminv" class="soft_anch"><code>harminv</code></a></td>
<td align="left">-&nbsp;<code>1.4.1</code></td>
<td align="left"><a href="https://github.com/stevengj/harminv">Website</a></td>
<td align="left">harminv is a program designed to solve the problem of harmonic inversion: given a time series consisting of a sum of sinusoids (modes), extract their frequencies and amplitudes.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="imkl" class="soft_anch"><code>imkl</code></a></td>
<td align="left">-&nbsp;<code>2017.u2</code><br/>-&nbsp;<code>2018.u1</code><br/>-&nbsp;<code>2018</code><br/>-&nbsp;<code>2019</code></td>
<td align="left"><a href="https://software.intel.com/en-us/intel-mkl">Website</a></td>
<td align="left">Intel Math Kernel Library (Intel MKL) is a library of optimized math routines for science, engineering, and financial applications. Core math functions include BLAS, LAPACK, ScaLAPACK, sparse solvers, fast Fourier transforms, and vector math.[3] The routines in MKL are hand-optimized specifically for Intel processors</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="libxsmm" class="soft_anch"><code>libxsmm</code></a></td>
<td align="left">-&nbsp;<code>1.8.1</code></td>
<td align="left"><a href="https://github.com/hfp/libxsmm">Website</a></td>
<td align="left">LIBXSMM is a library for small dense and small sparse matrix-matrix multiplications as well as for deep learning primitives such as small convolutions</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="metis" class="soft_anch"><code>metis</code></a></td>
<td align="left">-&nbsp;<code>5.1.0</code></td>
<td align="left"><a href="http://glaros.dtc.umn.edu/gkhome/metis/metis/overview">Website</a></td>
<td align="left">METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="mpfr" class="soft_anch"><code>mpfr</code></a></td>
<td align="left">-&nbsp;<code>3.1.5</code></td>
<td align="left"><a href="https://www.mpfr.org">Website</a></td>
<td align="left">The MPFR library is a C library for multiple-precision floating-point computations with correct rounding.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="mumps" class="soft_anch"><code>mumps</code></a></td>
<td align="left">-&nbsp;<code>5.1.2</code></td>
<td align="left"><a href="http://mumps.enseeiht.fr/">Website</a></td>
<td align="left">A parallel sparse direct solver.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="nagcl" class="soft_anch"><code>nagcl</code></a></td>
<td align="left">-&nbsp;<code>cll6i26dcl</code></td>
<td align="left"><a href="https://www.nag.com/nag-c-library">Website</a></td>
<td align="left">The NAG C Library is the largest and most comprehensive collection of mathematical and statistical algorithms for C and C++.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="nagfl" class="soft_anch"><code>nagfl</code></a></td>
<td align="left">-&nbsp;<code>fll6i26dcl</code></td>
<td align="left"><a href="https://www.nag.com/nag-fortran-library">Website</a></td>
<td align="left">The NAG Fortran Library is the largest and most comprehensive collection of numerical and statistical algorithms in Fortran.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="nagfs" class="soft_anch"><code>nagfs</code></a></td>
<td align="left">-&nbsp;<code>fsl6i26dcl</code></td>
<td align="left"><a href="https://www.nag.com/nag-library-smp-multicore">Website</a></td>
<td align="left">The NAG Library for SMP &amp; Multicore is based on, and includes, the full functionality of the NAG Fortran Library.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="nagmb" class="soft_anch"><code>nagmb</code></a></td>
<td align="left">-&nbsp;<code>MBL6I25DNL</code></td>
<td align="left"><a href="https://www.nag.com/nag-toolbox-matlab">Website</a></td>
<td align="left">The NAG C Library is the largest and most comprehensive collection of mathematical and statistical algorithms for C and C++.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="openblas" class="soft_anch"><code>openblas</code></a></td>
<td align="left">-&nbsp;<code>0.2.19</code><br/>-&nbsp;<code>0.3.4</code></td>
<td align="left"><a href="http://www.openblas.net/">Website</a></td>
<td align="left">OpenBLAS is an optimized BLAS library</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="parmetis" class="soft_anch"><code>parmetis</code></a></td>
<td align="left">-&nbsp;<code>4.0.3</code></td>
<td align="left"><a href="http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview">Website</a></td>
<td align="left">ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs, meshes, and for computing fill-reducing orderings of sparse matrices.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="py-numpy" class="soft_anch"><code>py-numpy</code></a></td>
<td align="left">-&nbsp;<code>1.14.3_py36</code><br/>-&nbsp;<code>1.14.3_py27</code></td>
<td align="left"><a href="http://www.numpy.org">Website</a></td>
<td align="left">NumPy is the fundamental package for scientific computing with Python.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="py-pyublas" class="soft_anch"><code>py-pyublas</code></a></td>
<td align="left">-&nbsp;<code>2017.1_py27</code></td>
<td align="left"><a href="https://mathema.tician.de/software/pyublas/">Website</a></td>
<td align="left">PyUblas provides a seamless glue layer between Numpy and Boost.Ublas for use with Boost.Python.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="py-scipy" class="soft_anch"><code>py-scipy</code></a></td>
<td align="left">-&nbsp;<code>1.1.0_py36</code><br/>-&nbsp;<code>1.1.0_py27</code></td>
<td align="left"><a href="https://www.scipy.org/scipylib">Website</a></td>
<td align="left">The SciPy library provides many user-friendly and efficient numerical routines such as routines for numerical integration and optimization.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="qrupdate" class="soft_anch"><code>qrupdate</code></a></td>
<td align="left">-&nbsp;<code>1.1.2</code></td>
<td align="left"><a href="https://sourceforge.net/projects/qrupdate/">Website</a></td>
<td align="left">qrupdate is a Fortran library for fast updates of QR and Cholesky decompositions.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="scalapack" class="soft_anch"><code>scalapack</code></a></td>
<td align="left">-&nbsp;<code>2.0.2</code></td>
<td align="left"><a href="http://www.netlib.org/scalapack/">Website</a></td>
<td align="left">ScaLAPACK is a library of high-performance linear algebra routines for parallel distributed memory machines.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="scotch" class="soft_anch"><code>scotch</code></a></td>
<td align="left">-&nbsp;<code>6.0.4</code></td>
<td align="left"><a href="http://www.labri.fr/perso/pelegrin/scotch">Website</a></td>
<td align="left">Software package and libraries for sequential and parallel graph partitioning, static mapping and clustering, sequential mesh and hypergraph partitioning, and sequential and parallel sparse matrix block ordering.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="superlu" class="soft_anch"><code>superlu</code></a></td>
<td align="left">-&nbsp;<code>5.2.1</code></td>
<td align="left"><a href="https://github.com/xiaoyeli/superlu">Website</a></td>
<td align="left">SuperLU is a general purpose library for the direct solution of large, sparse, nonsymmetric systems of linear equations.</td>
</tr>
<tr>
<td align="left"><strong>numerical library</strong></td>
<td align="left"><a id="xblas" class="soft_anch"><code>xblas</code></a></td>
<td align="left">-&nbsp;<code>1.0.248</code></td>
<td align="left"><a href="http://www.netlib.org/xblas">Website</a></td>
<td align="left">Extra precise basic linear algebra subroutines.</td>
</tr>
<tr>
<td align="left"><strong>optimization</strong></td>
<td align="left"><a id="gurobi" class="soft_anch"><code>gurobi</code></a></td>
<td align="left">-&nbsp;<code>7.5.1</code><br/>-&nbsp;<code>8.0.1_py27</code><br/>-&nbsp;<code>8.0.1_py36</code></td>
<td align="left"><a href="http://www.gurobi.com">Website</a></td>
<td align="left">The Gurobi Optimizer is a commercial optimization solver for mathematical programming.</td>
</tr>
<tr>
<td align="left"><strong>optimization</strong></td>
<td align="left"><a id="knitro" class="soft_anch"><code>knitro</code></a></td>
<td align="left">-&nbsp;<code>10.3.0</code></td>
<td align="left"><a href="https://www.artelys.com/en/optimization-tools/knitro">Website</a></td>
<td align="left">Artelys Knitro is an optimization solver for difficult large-scale nonlinear problems.</td>
</tr>
<tr>
<td align="left"><strong>scientific computing</strong></td>
<td align="left"><a id="py-scipystack" class="soft_anch"><code>py-scipystack</code></a></td>
<td align="left">-&nbsp;<code>1.0_py36</code><br/>-&nbsp;<code>1.0_py27</code></td>
<td align="left"><a href="https://www.scipy.org">Website</a></td>
<td align="left">The SciPy Stack is a collection of open source software for scientific computing in Python. It provides the following packages: numpy, scipy, matplotlib, ipython, jupyter, pandas, sympy and nose.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="R" class="soft_anch"><code>R</code></a></td>
<td align="left">-&nbsp;<code>3.4.0</code><br/>-&nbsp;<code>3.5.1</code></td>
<td align="left"><a href="http://r-project.org">Website</a></td>
<td align="left">R is a free software environment for statistical computing and graphics. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/R">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="datamash" class="soft_anch"><code>datamash</code></a></td>
<td align="left">-&nbsp;<code>1.3</code></td>
<td align="left"><a href="https://www.gnu.org/software/datamash/">Website</a></td>
<td align="left">GNU datamash is a command-line program which performs basic numeric, textual and statistical operations on input textual data files.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="jags" class="soft_anch"><code>jags</code></a></td>
<td align="left">-&nbsp;<code>4.3.0</code></td>
<td align="left"><a href="http://mcmc-jags.sourceforge.net/">Website</a></td>
<td align="left">Just another Gibbs sampler (JAGS) is a program for simulation from Bayesian hierarchical models using Markov chain Monte Carlo (MCMC).</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="py-rpy2" class="soft_anch"><code>py-rpy2</code></a></td>
<td align="left">-&nbsp;<code>2.8.6_py27</code><br/>-&nbsp;<code>2.9.2_py36</code></td>
<td align="left"><a href="https://rpy2.bitbucket.io">Website</a></td>
<td align="left">rpy2 is an interface to R running embedded in a Python process.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="r-rstan" class="soft_anch"><code>r-rstan</code></a></td>
<td align="left">-&nbsp;<code>2.17.3</code></td>
<td align="left"><a href="http://mc-stan.org/users/interfaces/rstan">Website</a></td>
<td align="left">RStan is the R interface to Stan, an open-source software for facilitating statistical inference at the frontiers of applied statistics.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="rstudio" class="soft_anch"><code>rstudio</code></a></td>
<td align="left">-&nbsp;<code>1.1.423</code></td>
<td align="left"><a href="http://www.rstudio.com">Website</a></td>
<td align="left">RStudio is an integrated development environment (IDE) for R. It includes a console, syntax-highlighting editor that supports direct code execution, as well as tools for plotting, history, debugging and workspace management.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="sas" class="soft_anch"><code>sas</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>9.4</code></td>
<td align="left"><a href="http://www.sas.com/">Website</a></td>
<td align="left">SAS is a software suite developed by SAS Institute for advanced analytics, multivariate analyses, business intelligence, data management, and predictive analytics.</td>
</tr>
<tr>
<td align="left"><strong>statistics</strong></td>
<td align="left"><a id="stata" class="soft_anch"><code>stata</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>14</code><br/>-&nbsp;<code>15</code></td>
<td align="left"><a href="https://www.stata.com">Website</a></td>
<td align="left">Stata is a complete, integrated statistical software package that provides everything you need for data analysis, data management, and graphics.</td>
</tr>
<tr>
<td align="left"><strong>symbolic</strong></td>
<td align="left"><a id="libmatheval" class="soft_anch"><code>libmatheval</code></a></td>
<td align="left">-&nbsp;<code>1.1.11</code></td>
<td align="left"><a href="https://www.gnu.org/software/libmatheval/">Website</a></td>
<td align="left">GNU libmatheval is a library (callable from C and Fortran) to parse and evaluate symbolic expressions input as text.</td>
</tr>
<tr>
<td align="left"><strong>symbolic</strong></td>
<td align="left"><a id="py-sympy" class="soft_anch"><code>py-sympy</code></a></td>
<td align="left">-&nbsp;<code>1.1.1_py27</code><br/>-&nbsp;<code>1.1.1_py36</code></td>
<td align="left"><a href="http://www.sympy.org">Website</a></td>
<td align="left">SymPy is a Python library for symbolic mathematics.</td>
</tr>
</tbody>
</table>
<h3 id="physics">Physics<a class="headerlink" href="#physics" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>astronomy</strong></td>
<td align="left"><a id="heasoft" class="soft_anch"><code>heasoft</code></a></td>
<td align="left">-&nbsp;<code>6.22.1</code></td>
<td align="left"><a href="https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/">Website</a></td>
<td align="left">HEAsoft is a Unified Release of the FTOOLS (General and mission-specific tools to manipulate FITS files) and XANADU (High-level, multi-mission tasks for X-ray astronomical spectral, timing, and imaging data analysis) software packages.</td>
</tr>
<tr>
<td align="left"><strong>geophysics</strong></td>
<td align="left"><a id="opensees" class="soft_anch"><code>opensees</code></a></td>
<td align="left">-&nbsp;<code>2.5.0</code></td>
<td align="left"><a href="http://opensees.berkeley.edu/">Website</a></td>
<td align="left">OpenSees is a software framework for developing applications to simulate the performance of structural and geotechnical systems subjected to earthquakes.</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="gdal" class="soft_anch"><code>gdal</code></a></td>
<td align="left">-&nbsp;<code>2.2.1</code></td>
<td align="left"><a href="http://www.gdal.org">Website</a></td>
<td align="left">GDAL is a translator library for raster and vector geospatial data formats.</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="geos" class="soft_anch"><code>geos</code></a></td>
<td align="left">-&nbsp;<code>3.6.2</code></td>
<td align="left"><a href="https://trac.osgeo.org/geos">Website</a></td>
<td align="left">GEOS (Geometry Engine - Open Source) is a C++ port of Java Topology Suite (JTS).</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="proj" class="soft_anch"><code>proj</code></a></td>
<td align="left">-&nbsp;<code>4.9.3</code></td>
<td align="left"><a href="http://proj4.org">Website</a></td>
<td align="left">proj.4 is a standard UNIX filter function which converts geographic longitude and latitude coordinates into cartesian coordinates (and vice versa.</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="py-opendrift" class="soft_anch"><code>py-opendrift</code></a></td>
<td align="left">-&nbsp;<code>1.0.3_py27</code></td>
<td align="left"><a href="https://github.com/OpenDrift/opendrift">Website</a></td>
<td align="left">OpenDrift is a software for modeling the trajectories and fate of objects or substances drifting in the ocean, or even in the atmosphere.</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="py-pyproj" class="soft_anch"><code>py-pyproj</code></a></td>
<td align="left">-&nbsp;<code>1.9.5.1_py36</code><br/>-&nbsp;<code>1.9.5.1_py27</code></td>
<td align="left"><a href="https://github.com/jswhit/pyproj">Website</a></td>
<td align="left">Python interface to PROJ4 library for cartographic transformations.</td>
</tr>
<tr>
<td align="left"><strong>geoscience</strong></td>
<td align="left"><a id="udunits" class="soft_anch"><code>udunits</code></a></td>
<td align="left">-&nbsp;<code>2.2.26</code></td>
<td align="left"><a href="https://www.unidata.ucar.edu/software/udunits/">Website</a></td>
<td align="left">The UDUNITS package from Unidata is a C-based package for the programatic handling of units of physical quantities.</td>
</tr>
<tr>
<td align="left"><strong>materials science</strong></td>
<td align="left"><a id="atat" class="soft_anch"><code>atat</code></a></td>
<td align="left">-&nbsp;<code>3.36</code></td>
<td align="left"><a href="https://www.brown.edu/Departments/Engineering/Labs/avdw/atat/">Website</a></td>
<td align="left">Alloy Theoretic Automated Toolkit: a software toolkit for modeling coupled configurational and vibrational disorder in alloy systems.</td>
</tr>
<tr>
<td align="left"><strong>photonics</strong></td>
<td align="left"><a id="meep" class="soft_anch"><code>meep</code></a></td>
<td align="left">-&nbsp;<code>1.3</code><br/>-&nbsp;<code>1.4.3</code></td>
<td align="left"><a href="https://github.com/stevengj/meep">Website</a></td>
<td align="left">Meep is a free finite-difference time-domain (FDTD) simulation software package to model electromagnetic systems.</td>
</tr>
<tr>
<td align="left"><strong>photonics</strong></td>
<td align="left"><a id="mpb" class="soft_anch"><code>mpb</code></a></td>
<td align="left">-&nbsp;<code>1.5</code><br/>-&nbsp;<code>1.6.2</code></td>
<td align="left"><a href="https://github.com/stevengj/mpb">Website</a></td>
<td align="left">MPB is a free software package for computing the band structures, or dispersion relations, and electromagnetic modes of periodic dielectric structures, on both serial and parallel computers.</td>
</tr>
</tbody>
</table>
<h3 id="biology">Biology<a class="headerlink" href="#biology" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>clinical science</strong></td>
<td align="left"><a id="simvascular" class="soft_anch"><code>simvascular</code></a></td>
<td align="left">-&nbsp;<code>20180704</code></td>
<td align="left"><a href="https://simvascular.github.io">Website</a></td>
<td align="left">Simvascular is a blood flow simulation and analysis toolkit. This module provides the svFSI (Fluid Solid Interaction) solver.</td>
</tr>
<tr>
<td align="left"><strong>computational biology</strong></td>
<td align="left"><a id="imp" class="soft_anch"><code>imp</code></a></td>
<td align="left">-&nbsp;<code>2.8.0</code></td>
<td align="left"><a href="https://integrativemodeling.org/">Website</a></td>
<td align="left">IMP's broad goal is to contribute to a comprehensive structural characterization of biomolecules ranging in size and complexity from small peptides to large macromolecular assemblies, by integrating data from diverse biochemical and biophysical experiments.</td>
</tr>
<tr>
<td align="left"><strong>computational biology</strong></td>
<td align="left"><a id="py-biopython" class="soft_anch"><code>py-biopython</code></a></td>
<td align="left">-&nbsp;<code>1.70_py27</code></td>
<td align="left"><a href="http://biopython.org">Website</a></td>
<td align="left">Biopython is a set of freely available tools for biological computation written in Python.</td>
</tr>
<tr>
<td align="left"><strong>computational biology</strong></td>
<td align="left"><a id="rosetta" class="soft_anch"><code>rosetta</code></a></td>
<td align="left">-&nbsp;<code>3.8</code></td>
<td align="left"><a href="https://www.rosettacommons.org">Website</a></td>
<td align="left">Rosetta is the premier software suite for modeling macromolecular structures. As a flexible, multi-purpose application, it includes tools for structure prediction, design, and remodeling of proteins and nucleic acids.</td>
</tr>
<tr>
<td align="left"><strong>cryo-em</strong></td>
<td align="left"><a id="eman2" class="soft_anch"><code>eman2</code></a></td>
<td align="left">-&nbsp;<code>2.2</code></td>
<td align="left"><a href="http://blake.bcm.edu/emanwiki/EMAN2">Website</a></td>
<td align="left">EMAN2 is a broadly based greyscale scientific image processing suite with a primary focus on processing data from transmission electron microscopes.</td>
</tr>
<tr>
<td align="left"><strong>cryo-em</strong></td>
<td align="left"><a id="relion" class="soft_anch"><code>relion</code></a></td>
<td align="left">-&nbsp;<code>2.0.3</code><br/>-&nbsp;<code>2.1</code></td>
<td align="left"><a href="http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page">Website</a></td>
<td align="left">RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class averages in electron cryo-microscopy (cryo-EM).</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="angsd" class="soft_anch"><code>angsd</code></a></td>
<td align="left">-&nbsp;<code>0.919</code></td>
<td align="left"><a href="http://www.popgen.dk/angsd/index.php/ANGSD">Website</a></td>
<td align="left">ANGSD is a software for analyzing next generation sequencing data.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="augustus" class="soft_anch"><code>augustus</code></a></td>
<td align="left">-&nbsp;<code>3.3.2</code></td>
<td align="left"><a href="https://github.com/Gaius-Augustus/Augustus">Website</a></td>
<td align="left">AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bamtools" class="soft_anch"><code>bamtools</code></a></td>
<td align="left">-&nbsp;<code>2.5.1</code></td>
<td align="left"><a href="https://github.com/pezmaster31/bamtools">Website</a></td>
<td align="left">BamTools is a project that provides both a C++ API and a command-line toolkit for reading, writing, and manipulating BAM (genome alignment) files.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bcftools" class="soft_anch"><code>bcftools</code></a></td>
<td align="left">-&nbsp;<code>1.6</code><br/>-&nbsp;<code>1.8</code></td>
<td align="left"><a href="https://github.com/samtools/bcftools">Website</a></td>
<td align="left">BCFtools is a program for variant calling and manipulating files in the Variant Call Format (VCF) and its binary counterpart BCF.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bcl2fastq" class="soft_anch"><code>bcl2fastq</code></a></td>
<td align="left">-&nbsp;<code>2.20</code></td>
<td align="left"><a href="https://support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html">Website</a></td>
<td align="left">The bcl2fastq2 conversion software can be used to convert BCL files from MiniSeq, MiSeq, NextSeq, HiSeq, iSeq and NovaSeq sequening systems.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bedtools" class="soft_anch"><code>bedtools</code></a></td>
<td align="left">-&nbsp;<code>2.27.1</code></td>
<td align="left"><a href="https://bedtools.readthedocs.io">Website</a></td>
<td align="left">The bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bowtie" class="soft_anch"><code>bowtie</code></a></td>
<td align="left">-&nbsp;<code>1.2.2</code></td>
<td align="left"><a href="http://bowtie-bio.sourceforge.net/">Website</a></td>
<td align="left">Bowtie is an ultrafast, memory-efficient short read aligner.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bowtie2" class="soft_anch"><code>bowtie2</code></a></td>
<td align="left">-&nbsp;<code>2.3.4.1</code></td>
<td align="left"><a href="http://bowtie-bio.sourceforge.net/bowtie2">Website</a></td>
<td align="left">Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="bwa" class="soft_anch"><code>bwa</code></a></td>
<td align="left">-&nbsp;<code>0.7.17</code></td>
<td align="left"><a href="http://bio-bwa.sourceforge.net/">Website</a></td>
<td align="left">BWA (Burrows-Wheeler Aligner) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="cufflinks" class="soft_anch"><code>cufflinks</code></a></td>
<td align="left">-&nbsp;<code>2.2.1</code></td>
<td align="left"><a href="https://cole-trapnell-lab.github.io/cufflinks/">Website</a></td>
<td align="left">Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="fastqc" class="soft_anch"><code>fastqc</code></a></td>
<td align="left">-&nbsp;<code>0.11.8</code></td>
<td align="left"><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">Website</a></td>
<td align="left">FastQC aims to provide a simple way to do some quality control checks on raw sequence data coming from high throughput sequencing pipelines.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="fastx_toolkit" class="soft_anch"><code>fastx_toolkit</code></a></td>
<td align="left">-&nbsp;<code>0.0.14</code></td>
<td align="left"><a href="http://hannonlab.cshl.edu/fastx_toolkit/">Website</a></td>
<td align="left">The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="freebayes" class="soft_anch"><code>freebayes</code></a></td>
<td align="left">-&nbsp;<code>1.2.0</code></td>
<td align="left"><a href="https://github.com/ekg/freebayes">Website</a></td>
<td align="left">FreeBayes is a Bayesian genetic variant detector designed to find small polymorphisms.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="gatk" class="soft_anch"><code>gatk</code></a></td>
<td align="left">-&nbsp;<code>4.1.0.0</code></td>
<td align="left"><a href="https://software.broadinstitute.org/gatk/">Website</a></td>
<td align="left">GATK (Genome Analysis Toolkit) offers a wide variety of tools with a primary focus on variant discovery and genotyping.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="hic-pro" class="soft_anch"><code>hic-pro</code></a></td>
<td align="left">-&nbsp;<code>2.10.0</code></td>
<td align="left"><a href="https://github.com/nservant/HiC-Pro">Website</a></td>
<td align="left">HiC-Pro: An optimized and flexible pipeline for Hi-C data processing.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="hisat2" class="soft_anch"><code>hisat2</code></a></td>
<td align="left">-&nbsp;<code>2.1.0</code></td>
<td align="left"><a href="http://ccb.jhu.edu/software/hisat2">Website</a></td>
<td align="left">HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes (as well as to a single reference genome).</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="htslib" class="soft_anch"><code>htslib</code></a></td>
<td align="left">-&nbsp;<code>1.6</code><br/>-&nbsp;<code>1.8</code></td>
<td align="left"><a href="https://github.com/samtools/htslib">Website</a></td>
<td align="left">C library for high-throughput sequencing data formats.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="jellyfish" class="soft_anch"><code>jellyfish</code></a></td>
<td align="left">-&nbsp;<code>2.2.10</code></td>
<td align="left"><a href="https://github.com/gmarcais/Jellyfish">Website</a></td>
<td align="left">A fast multi-threaded k-mer counter.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="kallisto" class="soft_anch"><code>kallisto</code></a></td>
<td align="left">-&nbsp;<code>0.44.0</code></td>
<td align="left"><a href="https://pachterlab.github.io/kallisto/">Website</a></td>
<td align="left">kallisto is a program for quantifying abundances of transcripts from RNA-Seq data using high-throughput sequencing reads.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="mixcr" class="soft_anch"><code>mixcr</code></a></td>
<td align="left">-&nbsp;<code>2.1.12</code></td>
<td align="left"><a href="https://milaboratory.com/software/mixcr/">Website</a></td>
<td align="left">MiXCR is a universal framework that processes big immunome data from raw sequences to quantitated clonotypes.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="ncbi-blast+" class="soft_anch"><code>ncbi-blast+</code></a></td>
<td align="left">-&nbsp;<code>2.6.0</code><br/>-&nbsp;<code>2.7.1</code></td>
<td align="left"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">Website</a></td>
<td align="left">NCBI BLAST+ is a suite of command-line tools to run BLAST (Basic Local Alignment Search Tool), an algorithm for comparing primary biological sequence information.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="plink" class="soft_anch"><code>plink</code></a></td>
<td align="left">-&nbsp;<code>1.07</code><br/>-&nbsp;<code>1.90b5.3</code><br/>-&nbsp;<code>2.0a1</code><br/>-&nbsp;<code>2.0a2</code></td>
<td align="left"><a href="http://zzz.bwh.harvard.edu/plink/">Website</a></td>
<td align="left">PLINK is a free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-busco" class="soft_anch"><code>py-busco</code></a></td>
<td align="left">-&nbsp;<code>3.0.2_py27</code></td>
<td align="left"><a href="https://busco.ezlab.org/">Website</a></td>
<td align="left">Assessing genome assembly and annotation completeness with Benchmarking Universal Single-Copy Orthologs (BUSCO).</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-bx-python" class="soft_anch"><code>py-bx-python</code></a></td>
<td align="left">-&nbsp;<code>0.8.1_py27</code></td>
<td align="left"><a href="https://github.com/bxlab/bx-python">Website</a></td>
<td align="left">Tools for manipulating biological data, particularly multiple sequence alignments.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-cutadapt" class="soft_anch"><code>py-cutadapt</code></a></td>
<td align="left">-&nbsp;<code>1.18_py36</code><br/>-&nbsp;<code>1.18_py27</code></td>
<td align="left"><a href="https://cutadapt.readthedocs.io">Website</a></td>
<td align="left">Cutadapt finds and removes adapter sequences, primers, poly-A tails and other types of unwanted sequence from your high-throughput sequencing reads.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-fithic" class="soft_anch"><code>py-fithic</code></a></td>
<td align="left">-&nbsp;<code>1.1.3_py27</code></td>
<td align="left"><a href="https://github.com/ay-lab/fithic">Website</a></td>
<td align="left">Fit-Hi-C is a tool for assigning statistical confidence estimates to chromosomal contact maps produced by genome architecture assays.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-macs2" class="soft_anch"><code>py-macs2</code></a></td>
<td align="left">-&nbsp;<code>2.1.1_py27</code></td>
<td align="left"><a href="https://github.com/taoliu/MACS">Website</a></td>
<td align="left">MACS (Model-based Analysis of ChIP-Seq) implements a novel ChIP-Seq analysis method.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-multiqc" class="soft_anch"><code>py-multiqc</code></a></td>
<td align="left">-&nbsp;<code>1.6_py36</code><br/>-&nbsp;<code>1.6_py27</code></td>
<td align="left"><a href="https://multiqc.info">Website</a></td>
<td align="left">MultiQC is a reporting tool that parses summary statistics from results and log files generated by other bioinformatics tools.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-pybedtools" class="soft_anch"><code>py-pybedtools</code></a></td>
<td align="left">-&nbsp;<code>0.8.0_py27</code></td>
<td align="left"><a href="https://github.com/daler/pybedtools">Website</a></td>
<td align="left">Pybedtools wraps and extends BEDTools and offers feature-level manipulations from within Python.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="py-pysam" class="soft_anch"><code>py-pysam</code></a></td>
<td align="left">-&nbsp;<code>0.14.1_py27</code></td>
<td align="left"><a href="https://github.com/pysam-developers/pysam">Website</a></td>
<td align="left">Pysam is a python module for reading, manipulating and writing genomic data sets.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="salmon" class="soft_anch"><code>salmon</code></a></td>
<td align="left">-&nbsp;<code>0.12.0</code></td>
<td align="left"><a href="https://combine-lab.github.io/salmon">Website</a></td>
<td align="left">Highly-accurate &amp; wicked fast transcript-level quantification from RNA-seq reads using lightweight alignments.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="samtools" class="soft_anch"><code>samtools</code></a></td>
<td align="left">-&nbsp;<code>1.6</code><br/>-&nbsp;<code>1.8</code></td>
<td align="left"><a href="https://github.com/samtools/samtools">Website</a></td>
<td align="left">Tools (written in C using htslib) for manipulating next-generation sequencing data.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="sentieon" class="soft_anch"><code>sentieon</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>201808.01</code></td>
<td align="left"><a href="https://www.sentieon.com/">Website</a></td>
<td align="left">Sentieon Genomics software is a set of software tools that perform analysis of genomic data obtained from DNA sequencing.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="star" class="soft_anch"><code>star</code></a></td>
<td align="left">-&nbsp;<code>2.5.4b</code></td>
<td align="left"><a href="https://github.com/alexdobin/STAR">Website</a></td>
<td align="left">STAR: ultrafast universal RNA-seq aligner.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="tophat" class="soft_anch"><code>tophat</code></a></td>
<td align="left">-&nbsp;<code>2.1.1</code></td>
<td align="left"><a href="http://ccb.jhu.edu/software/tophat/index.shtml">Website</a></td>
<td align="left">TopHat is a fast splice junction mapper for RNA-Seq reads.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="trim_galore" class="soft_anch"><code>trim_galore</code></a></td>
<td align="left">-&nbsp;<code>0.5.0</code></td>
<td align="left"><a href="https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/">Website</a></td>
<td align="left">Trim Galore! is a wrapper script to automate quality and adapter trimming as well as quality control, with some added functionality to remove biased methylation positions for RRBS sequence files.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="trinity" class="soft_anch"><code>trinity</code></a></td>
<td align="left">-&nbsp;<code>2.8.4</code></td>
<td align="left"><a href="http://trinityrnaseq.github.io">Website</a></td>
<td align="left">Trinity RNA-Seq de novo transcriptome assembly.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="vcflib" class="soft_anch"><code>vcflib</code></a></td>
<td align="left">-&nbsp;<code>1.0.0</code></td>
<td align="left"><a href="https://github.com/vcflib/vcflib">Website</a></td>
<td align="left">A C++ library for parsing and manipulating VCF files.</td>
</tr>
<tr>
<td align="left"><strong>genomics</strong></td>
<td align="left"><a id="vcftools" class="soft_anch"><code>vcftools</code></a></td>
<td align="left">-&nbsp;<code>0.1.15</code></td>
<td align="left"><a href="https://vcftools.github.io">Website</a></td>
<td align="left">VCFtools is a program package designed for working with VCF files, such as those generated by the 1000 Genomes Project.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="afni" class="soft_anch"><code>afni</code></a></td>
<td align="left">-&nbsp;<code>17.2.07</code><br/>-&nbsp;<code>18.2.04</code></td>
<td align="left"><a href="https://afni.nimh.nih.gov">Website</a></td>
<td align="left">AFNI (Analysis of Functional NeuroImages) is a set of C programs for processing, analyzing, and displaying functional MRI (FMRI) data - a technique for mapping human brain activity.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="ants" class="soft_anch"><code>ants</code></a></td>
<td align="left">-&nbsp;<code>2.1.0</code></td>
<td align="left"><a href="https://stnava.github.io/ANTs">Website</a></td>
<td align="left">ANTs computes high-dimensional mappings to capture the statistics of brain structure and function.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="dcm2niix" class="soft_anch"><code>dcm2niix</code></a></td>
<td align="left">-&nbsp;<code>1.0.20171215</code></td>
<td align="left"><a href="https://github.com/rordenlab/dcm2niix">Website</a></td>
<td align="left">dcm2niix is a program esigned to convert neuroimaging data from the DICOM format to the NIfTI format.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="freesurfer" class="soft_anch"><code>freesurfer</code></a></td>
<td align="left">-&nbsp;<code>6.0.0</code></td>
<td align="left"><a href="https://surfer.nmr.mgh.harvard.edu/">Website</a></td>
<td align="left">An open source software suite for processing and analyzing (human) brain MRI images.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="fsl" class="soft_anch"><code>fsl</code></a></td>
<td align="left">-&nbsp;<code>5.0.10</code></td>
<td align="left"><a href="https://fsl.fmrib.ox.ac.uk/fsl">Website</a></td>
<td align="left">FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="mricron" class="soft_anch"><code>mricron</code></a></td>
<td align="left">-&nbsp;<code>20160502</code></td>
<td align="left"><a href="https://www.nitrc.org/projects/mricron">Website</a></td>
<td align="left">MRIcron is a cross-platform NIfTI format image viewer.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="mrtrix" class="soft_anch"><code>mrtrix</code></a></td>
<td align="left">-&nbsp;<code>0.3.16</code></td>
<td align="left"><a href="http://www.mrtrix.org">Website</a></td>
<td align="left">MRtrix3 provides a set of tools to perform various types of diffusion MRI analyses, from various forms of tractography through to next-generation group-level analyses.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="py-caiman" class="soft_anch"><code>py-caiman</code></a></td>
<td align="left">-&nbsp;<code>1.4.4_py36</code></td>
<td align="left"><a href="https://github.com/flatironinstitute/CaImAn/">Website</a></td>
<td align="left">A Python toolbox for large scale Calcium Imaging data Analysis and behavioral analysis.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="py-mdt" class="soft_anch"><code>py-mdt</code></a></td>
<td align="left">-&nbsp;<code>0.10.9_py36</code></td>
<td align="left"><a href="https://github.com/cbclab/MDT">Website</a></td>
<td align="left">The Maastricht Diffusion Toolbox, MDT, is a framework and library for parallelized (GPU and multi-core CPU) diffusion Magnetic Resonance Imaging (MRI) modeling.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="py-nipype" class="soft_anch"><code>py-nipype</code></a></td>
<td align="left">-&nbsp;<code>1.1.3_py27</code><br/>-&nbsp;<code>1.1.3_py36</code></td>
<td align="left"><a href="https://nipype.readthedocs.io/en/latest/index.html">Website</a></td>
<td align="left">Nipype is a Python project that provides a uniform interface to existing neuroimaging software and facilitates interaction between these packages within a single workflow.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="spm" class="soft_anch"><code>spm</code></a></td>
<td align="left">-&nbsp;<code>12</code></td>
<td align="left"><a href="http://www.fil.ion.ucl.ac.uk/spm/">Website</a></td>
<td align="left">The SPM software package has been designed for the analysis of brain imaging data sequences. The sequences can be a series of images from different cohorts, or time-series from the same subject.</td>
</tr>
<tr>
<td align="left"><strong>neurology</strong></td>
<td align="left"><a id="workbench" class="soft_anch"><code>workbench</code></a></td>
<td align="left">-&nbsp;<code>1.3.1</code></td>
<td align="left"><a href="https://www.humanconnectome.org/software/connectome-workbench">Website</a></td>
<td align="left">Connectome Workbench is an open source, freely available visualization and discovery tool used to map neuroimaging data, especially data generated by the Human Connectome Project.</td>
</tr>
<tr>
<td align="left"><strong>pathology</strong></td>
<td align="left"><a id="openslide" class="soft_anch"><code>openslide</code></a></td>
<td align="left">-&nbsp;<code>3.4.1</code></td>
<td align="left"><a href="https://openslide.org/">Website</a></td>
<td align="left">OpenSlide is a C library that provides a simple interface to read whole-slide images (also known as virtual slides).</td>
</tr>
<tr>
<td align="left"><strong>pathology</strong></td>
<td align="left"><a id="py-openslide-python" class="soft_anch"><code>py-openslide-python</code></a></td>
<td align="left">-&nbsp;<code>1.1.1_py36</code><br/>-&nbsp;<code>1.1.1_py27</code></td>
<td align="left"><a href="https://openslide.org/api/python/">Website</a></td>
<td align="left">OpenSlide Python is a Python interface to the OpenSlide library.</td>
</tr>
<tr>
<td align="left"><strong>phylogenetics</strong></td>
<td align="left"><a id="py-ete" class="soft_anch"><code>py-ete</code></a></td>
<td align="left">-&nbsp;<code>3.0.0_py27</code></td>
<td align="left"><a href="http://etetoolkit.org/">Website</a></td>
<td align="left">A Python framework for the analysis and visualization of trees.</td>
</tr>
</tbody>
</table>
<h3 id="chemistry">Chemistry<a class="headerlink" href="#chemistry" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>cheminformatics</strong></td>
<td align="left"><a id="py-rdkit" class="soft_anch"><code>py-rdkit</code></a></td>
<td align="left">-&nbsp;<code>2018.09.1_py36</code><br/>-&nbsp;<code>2018.09.1_py27</code></td>
<td align="left"><a href="http://www.rdkit.org">Website</a></td>
<td align="left">RDKit is a collection of cheminformatics and machine-learning software written in C++ and Python.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="gaussian" class="soft_anch"><code>gaussian</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>g16</code></td>
<td align="left"><a href="https://www.gaussian.com">Website</a></td>
<td align="left">Gaussian is a general purpose computational chemistry software package.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="libint" class="soft_anch"><code>libint</code></a></td>
<td align="left">-&nbsp;<code>1.1.4</code><br/>-&nbsp;<code>2.0.3</code></td>
<td align="left"><a href="https://sourceforge.net/p/libint/home/">Website</a></td>
<td align="left">Libint computes molecular integrals.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="libxc" class="soft_anch"><code>libxc</code></a></td>
<td align="left">-&nbsp;<code>3.0.0</code></td>
<td align="left"><a href="http://octopus-code.org/wiki/Libxc">Website</a></td>
<td align="left">Libxc is a library of exchange-correlation functionals for density-functional theory.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="nwchem" class="soft_anch"><code>nwchem</code></a></td>
<td align="left">-&nbsp;<code>6.8</code></td>
<td align="left"><a href="http://www.nwchem-sw.org">Website</a></td>
<td align="left">NWChem is an ab initio computational chemistry software package which also includes quantum chemical and molecular dynamics functionality.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="py-ase" class="soft_anch"><code>py-ase</code></a></td>
<td align="left">-&nbsp;<code>3.14.1_py27</code></td>
<td align="left"><a href="https://wiki.fysik.dtu.dk/ase/">Website</a></td>
<td align="left">The Atomic Simulation Environment (ASE) is a set of tools and Python modules for setting up, manipulating, running, visualizing and analyzing atomistic simulations.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="schrodinger" class="soft_anch"><code>schrodinger</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>2017-3</code><br/>-&nbsp;<code>2018-1</code><br/>-&nbsp;<code>2018-2</code></td>
<td align="left"><a href="https://www.schrodinger.com">Website</a></td>
<td align="left">Schrdinger Suites (Small-molecule Drug Discovery Suite, Material Science Suite, Biologics Suite) provide a set of molecular modelling software.</td>
</tr>
<tr>
<td align="left"><strong>computational chemistry</strong></td>
<td align="left"><a id="vasp" class="soft_anch"><code>vasp</code></a><sup><b class="lic"></b></sup></td>
<td align="left">-&nbsp;<code>5.4.1</code></td>
<td align="left"><a href="https://www.vasp.at">Website</a></td>
<td align="left">The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics, from first principles.</td>
</tr>
<tr>
<td align="left"><strong>cristallography</strong></td>
<td align="left"><a id="vesta" class="soft_anch"><code>vesta</code></a></td>
<td align="left">-&nbsp;<code>3.4.4</code></td>
<td align="left"><a href="http://jp-minerals.org/vesta/en/">Website</a></td>
<td align="left">VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.</td>
</tr>
<tr>
<td align="left"><strong>electrostatics</strong></td>
<td align="left"><a id="apbs" class="soft_anch"><code>apbs</code></a></td>
<td align="left">-&nbsp;<code>1.5</code></td>
<td align="left"><a href="http://www.poissonboltzmann.org/">Website</a></td>
<td align="left">APBS solves the equations of continuum electrostatics for large biomolecular assemblages.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="gromacs" class="soft_anch"><code>gromacs</code></a></td>
<td align="left">-&nbsp;<code>2016.3</code><br/>-&nbsp;<code>2018</code></td>
<td align="left"><a href="http://www.gromacs.org">Website</a></td>
<td align="left">GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="lammps" class="soft_anch"><code>lammps</code></a></td>
<td align="left">-&nbsp;<code>20180316</code></td>
<td align="left"><a href="http://lammps.sandia.gov">Website</a></td>
<td align="left">LAMMPS is a classical molecular dynamics code that models an ensemble of particles in a liquid, solid, or gaseous state.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="openmm" class="soft_anch"><code>openmm</code></a></td>
<td align="left">-&nbsp;<code>7.1.1</code></td>
<td align="left"><a href="http://openmm.org">Website</a></td>
<td align="left">A high performance toolkit for molecular simulation.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="plumed" class="soft_anch"><code>plumed</code></a></td>
<td align="left">-&nbsp;<code>2.3.2</code></td>
<td align="left"><a href="http://www.plumed.org/">Website</a></td>
<td align="left">PLUMED is an open source library for free energy calculations in molecular systems.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="py-raspa2" class="soft_anch"><code>py-raspa2</code></a></td>
<td align="left">-&nbsp;<code>2.0.3_py27</code></td>
<td align="left"><a href="https://github.com/numat/RASPA2">Website</a></td>
<td align="left">RASPA2 is a general purpose classical simulation package that can be used for the simulation of molecules in gases, fluids, zeolites, aluminosilicates, metal-organic frameworks, carbon nanotubes and external fields.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="qbox" class="soft_anch"><code>qbox</code></a></td>
<td align="left">-&nbsp;<code>1.65.0</code></td>
<td align="left"><a href="http://www.qboxcode.org">Website</a></td>
<td align="left">Qbox is a First-Principles Molecular Dynamics code.</td>
</tr>
<tr>
<td align="left"><strong>molecular dynamics</strong></td>
<td align="left"><a id="quip" class="soft_anch"><code>quip</code></a></td>
<td align="left">-&nbsp;<code>20170901</code></td>
<td align="left"><a href="https://github.com/libAtoms/QUIP">Website</a></td>
<td align="left">The QUIP package is a collection of software tools to carry out molecular dynamics simulations.</td>
</tr>
<tr>
<td align="left"><strong>quantum chemistry</strong></td>
<td align="left"><a id="cp2k" class="soft_anch"><code>cp2k</code></a></td>
<td align="left">-&nbsp;<code>4.1</code></td>
<td align="left"><a href="http://www.cp2k.org">Website</a></td>
<td align="left">CP2K is a quantum chemistry and solid state physics software package that can perform atomistic simulations of solid state, liquid, molecular, periodic, material, crystal, and biological systems.</td>
</tr>
<tr>
<td align="left"><strong>quantum chemistry</strong></td>
<td align="left"><a id="quantum-espresso" class="soft_anch"><code>quantum-espresso</code></a></td>
<td align="left">-&nbsp;<code>6.2.1</code></td>
<td align="left"><a href="https://www.quantum-espresso.org">Website</a></td>
<td align="left">Quantum ESPRESSO is an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials.</td>
</tr>
<tr>
<td align="left"><strong>quantum chemistry</strong></td>
<td align="left"><a id="quantum-espresso_gpu" class="soft_anch"><code>quantum-espresso_gpu</code></a></td>
<td align="left">-&nbsp;<code>1.1</code></td>
<td align="left"><a href="https://github.com/fspiga/qe-gpu">Website</a></td>
<td align="left">GPU-accelerated Quantum ESPRESSO using CUDA FORTRAN</td>
</tr>
</tbody>
</table>
<h3 id="devel">Devel<a class="headerlink" href="#devel" title="Permanent link">#</a></h3>
<table>
<thead>
<tr>
<th align="left">Field</th>
<th align="left"><img style="float:left;min-width:110px;visibility:hidden">Module&nbsp;name</th>
<th align="left"><img style="float:left;min-width:90px;visibility:hidden">Version(s)</th>
<th align="left">URL</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left"><strong>build</strong></td>
<td align="left"><a id="bazel" class="soft_anch"><code>bazel</code></a></td>
<td align="left">-&nbsp;<code>0.16.1</code></td>
<td align="left"><a href="https://bazel.build/">Website</a></td>
<td align="left">Bazel is a fast, scalable, multi-language and extensible build system.</td>
</tr>
<tr>
<td align="left"><strong>build</strong></td>
<td align="left"><a id="cmake" class="soft_anch"><code>cmake</code></a></td>
<td align="left">-&nbsp;<code>3.8.1</code><br/>-&nbsp;<code>3.11.1</code><br/>-&nbsp;<code>3.13.1</code></td>
<td align="left"><a href="https://www.cmake.org">Website</a></td>
<td align="left">CMake is an extensible, open-source system that manages the build process in an operating system and in a compiler-independent manner.</td>
</tr>
<tr>
<td align="left"><strong>build</strong></td>
<td align="left"><a id="kerl" class="soft_anch"><code>kerl</code></a></td>
<td align="left">-&nbsp;<code>1.8.5</code></td>
<td align="left"><a href="https://github.com/kerl/kerl">Website</a></td>
<td align="left">Kerl is a tool to easily build and install Erlang/OTP instances.</td>
</tr>
<tr>
<td align="left"><strong>build</strong></td>
<td align="left"><a id="ninja" class="soft_anch"><code>ninja</code></a></td>
<td align="left">-&nbsp;<code>1.9.0</code></td>
<td align="left"><a href="https://ninja-build.org/">Website</a></td>
<td align="left">Ninja is a small build system with a focus on speed.</td>
</tr>
<tr>
<td align="left"><strong>build</strong></td>
<td align="left"><a id="scons" class="soft_anch"><code>scons</code></a></td>
<td align="left">-&nbsp;<code>2.5.1_py36</code><br/>-&nbsp;<code>2.5.1_py27</code></td>
<td align="left"><a href="http://www.scons.org">Website</a></td>
<td align="left">SCons is an Open Source software construction tool.</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="gcc" class="soft_anch"><code>gcc</code></a></td>
<td align="left">-&nbsp;<code>7.1.0</code><br/>-&nbsp;<code>7.3.0</code><br/>-&nbsp;<code>8.1.0</code><br/>-&nbsp;<code>9.1.0</code><br/>-&nbsp;<code>6.3.0</code></td>
<td align="left"><a href="http://gcc.gnu.org">Website</a></td>
<td align="left">The GNU Compiler Collection includes front ends for C, C++, Fortran, Java, and Go, as well as libraries for these languages (libstdc++, libgcj,...).</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="icc" class="soft_anch"><code>icc</code></a></td>
<td align="left">-&nbsp;<code>2017.u2</code><br/>-&nbsp;<code>2018.u1</code><br/>-&nbsp;<code>2018</code><br/>-&nbsp;<code>2019</code></td>
<td align="left"><a href="https://software.intel.com/en-us/c-compilers">Website</a></td>
<td align="left">Intel C++ Compiler, also known as icc or icl, is a group of C and C++ compilers from Intel</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="ifort" class="soft_anch"><code>ifort</code></a></td>
<td align="left">-&nbsp;<code>2017.u2</code><br/>-&nbsp;<code>2018.u1</code><br/>-&nbsp;<code>2018</code><br/>-&nbsp;<code>2019</code></td>
<td align="left"><a href="https://software.intel.com/en-us/fortran-compilers">Website</a></td>
<td align="left">Intel Fortran Compiler, also known as ifort, is a group of Fortran compilers from Intel</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="llvm" class="soft_anch"><code>llvm</code></a></td>
<td align="left">-&nbsp;<code>3.8.1</code><br/>-&nbsp;<code>4.0.0</code><br/>-&nbsp;<code>5.0.0</code><br/>-&nbsp;<code>7.0.0</code></td>
<td align="left"><a href="http://llvm.org">Website</a></td>
<td align="left">The LLVM Project is a collection of modular and reusable compiler and toolchain technologies. Clang is an LLVM native C/C++/Objective-C compiler,</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="nagfor" class="soft_anch"><code>nagfor</code></a></td>
<td align="left">-&nbsp;<code>npl6a61na</code></td>
<td align="left"><a href="https://www.nag.com/nag-compiler">Website</a></td>
<td align="left">The NAG Fortran Compiler is a full standard implementation of the ISO Fortran 95 language with the addition of all of Fortran 2003, most of Fortran 2008 and OpenMP 3.0 and 3.1.</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="pgi" class="soft_anch"><code>pgi</code></a></td>
<td align="left">-&nbsp;<code>17.4</code><br/>-&nbsp;<code>18.4</code></td>
<td align="left"><a href="http://pgroup.com">Website</a></td>
<td align="left">PGI compilers and tools, including Open MPI (Community Edition).</td>
</tr>
<tr>
<td align="left"><strong>compiler</strong></td>
<td align="left"><a id="smlnj" class="soft_anch"><code>smlnj</code></a></td>
<td align="left">-&nbsp;<code>110.81</code></td>
<td align="left"><a href="http://smlnj.org/">Website</a></td>
<td align="left">Standard ML of New Jersey (abbreviated SML/NJ) is a compiler for the Standard ML '97 programming language.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="h5utils" class="soft_anch"><code>h5utils</code></a></td>
<td align="left">-&nbsp;<code>1.12.1</code></td>
<td align="left"><a href="http://ab-initio.mit.edu/wiki/index.php/H5utils">Website</a></td>
<td align="left">h5utils is a set of utilities for visualization and conversion of scientific data in the free, portable HDF5 format.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="hdf5" class="soft_anch"><code>hdf5</code></a></td>
<td align="left">-&nbsp;<code>1.10.0p1</code><br/>-&nbsp;<code>1.10.2</code></td>
<td align="left"><a href="https://www.hdfgroup.org/hdf5">Website</a></td>
<td align="left">HDF5 is a data model, library, and file format for storing and managing data. It supports an unlimited variety of datatypes, and is designed for flexible and efficient I/O and for high volume and complex data.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="hiredis" class="soft_anch"><code>hiredis</code></a></td>
<td align="left">-&nbsp;<code>0.13.3</code></td>
<td align="left"><a href="https://github.com/redis/hiredis">Website</a></td>
<td align="left">Hiredis is a minimalistic C client library for the Redis database.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="ncl" class="soft_anch"><code>ncl</code></a></td>
<td align="left">-&nbsp;<code>6.4.0</code></td>
<td align="left"><a href="http://www.ncl.ucar.edu/">Website</a></td>
<td align="left">NCL is a free interpreted language designed specifically for scientific data processing and visualization.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="netcdf" class="soft_anch"><code>netcdf</code></a></td>
<td align="left">-&nbsp;<code>4.4.1.1</code></td>
<td align="left"><a href="https://www.unidata.ucar.edu/software/netcdf">Website</a></td>
<td align="left">NetCDF is a set of software libraries and self-describing, machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="pnetcdf" class="soft_anch"><code>pnetcdf</code></a></td>
<td align="left">-&nbsp;<code>1.8.1</code></td>
<td align="left"><a href="http://cucis.ece.northwestern.edu/projects/PnetCDF/">Website</a></td>
<td align="left">Parallel netCDF (PnetCDF) is a parallel I/O library for accessing NetCDF files in CDF-1, 2, and 5 formats.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="protobuf" class="soft_anch"><code>protobuf</code></a></td>
<td align="left">-&nbsp;<code>3.4.0</code></td>
<td align="left"><a href="https://developers.google.com/protocol-buffers/">Website</a></td>
<td align="left">Protocol Buffers (a.k.a., protobuf) are Google's language-neutral, platform-neutral, extensible mechanism for serializing structured data.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="py-pandas" class="soft_anch"><code>py-pandas</code></a></td>
<td align="left">-&nbsp;<code>0.23.0_py27</code><br/>-&nbsp;<code>0.23.0_py36</code></td>
<td align="left"><a href="https://pandas.pydata.org/">Website</a></td>
<td align="left">pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures and data analysis tools for the Python programming language.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="py-protobuf" class="soft_anch"><code>py-protobuf</code></a></td>
<td align="left">-&nbsp;<code>3.4.0_py36</code><br/>-&nbsp;<code>3.6.1_py27</code><br/>-&nbsp;<code>3.6.1_py36</code><br/>-&nbsp;<code>3.4.0_py27</code></td>
<td align="left"><a href="https://developers.google.com/protocol-buffers">Website</a></td>
<td align="left">Python bindings for Google's Protocol Buffers data interchange format.</td>
</tr>
<tr>
<td align="left"><strong>data</strong></td>
<td align="left"><a id="redis" class="soft_anch"><code>redis</code></a></td>
<td align="left">-&nbsp;<code>4.0.1</code></td>
<td align="left"><a href="https://redis.io">Website</a></td>
<td align="left">Redis is an open source, in-memory data structure store, used as a database, cache and message broker.</td>
</tr>
<tr>
<td align="left"><strong>data analytics</strong></td>
<td align="left"><a id="hadoop" class="soft_anch"><code>hadoop</code></a></td>
<td align="left">-&nbsp;<code>3.1.0</code></td>
<td align="left"><a href="https://hadoop.apache.org/">Website</a></td>
<td align="left">The Apache Hadoop software library is a framework that allows for the distributed processing of large data sets across clusters of computers using simple programming models.</td>
</tr>
<tr>
<td align="left"><strong>data analytics</strong></td>
<td align="left"><a id="py-sparkhpc" class="soft_anch"><code>py-sparkhpc</code></a></td>
<td align="left">-&nbsp;<code>0.3</code></td>
<td align="left"><a href="https://sparkhpc.readthedocs.io">Website</a></td>
<td align="left">Launching and controlling spark on HPC clusters</td>
</tr>
<tr>
<td align="left"><strong>data analytics</strong></td>
<td align="left"><a id="spark" class="soft_anch"><code>spark</code></a></td>
<td align="left">-&nbsp;<code>2.3.0</code></td>
<td align="left"><a href="https://spark.apache.org">Website</a></td>
<td align="left">Apache Spark is a unified analytics engine for large-scale data processing. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/spark">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>debug</strong></td>
<td align="left"><a id="gdb" class="soft_anch"><code>gdb</code></a></td>
<td align="left">-&nbsp;<code>8.2.1</code></td>
<td align="left"><a href="https://www.gnu.org/software/gdb/">Website</a></td>
<td align="left">GDB is the GNU Project debugger.</td>
</tr>
<tr>
<td align="left"><strong>debug</strong></td>
<td align="left"><a id="valgrind" class="soft_anch"><code>valgrind</code></a></td>
<td align="left">-&nbsp;<code>3.14.0</code></td>
<td align="left"><a href="http://valgrind.org/">Website</a></td>
<td align="left">Valgrind is an instrumentation framework for building dynamic analysis tools.</td>
</tr>
<tr>
<td align="left"><strong>framework</strong></td>
<td align="left"><a id="dotnet" class="soft_anch"><code>dotnet</code></a></td>
<td align="left">-&nbsp;<code>2.1.500</code></td>
<td align="left"><a href="https://dotnet.microsoft.com/">Website</a></td>
<td align="left">.NET is a free, cross-platform, open source developer platform for building many different types of applications.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="chapel" class="soft_anch"><code>chapel</code></a></td>
<td align="left">-&nbsp;<code>1.19.0</code></td>
<td align="left"><a href="https://chapel-lang.org/">Website</a></td>
<td align="left">Chapel is a modern programming language designed for productive parallel computing at scale.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="cuda" class="soft_anch"><code>cuda</code></a></td>
<td align="left">-&nbsp;<code>8.0.61</code><br/>-&nbsp;<code>9.1.85</code><br/>-&nbsp;<code>9.2.88</code><br/>-&nbsp;<code>9.2.148</code><br/>-&nbsp;<code>10.0.130</code><br/>-&nbsp;<code>10.1.105</code><br/>-&nbsp;<code>9.0.176</code></td>
<td align="left"><a href="https://developer.nvidia.com/cuda-toolkit">Website</a></td>
<td align="left">CUDA is a parallel computing platform and application programming interface (API) model created by Nvidia. It allows software developers and software engineers to use a CUDA-enabled graphics processing unit (GPU) for general purpose processing.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="erlang" class="soft_anch"><code>erlang</code></a></td>
<td align="left">-&nbsp;<code>21.3</code></td>
<td align="left"><a href="http://erlang.org/">Website</a></td>
<td align="left">Erlang is a programming language used to build massively scalable soft real-time systems with requirements on high availability.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="go" class="soft_anch"><code>go</code></a></td>
<td align="left">-&nbsp;<code>1.9</code></td>
<td align="left"><a href="http://golang.org">Website</a></td>
<td align="left">Go is an open source programming language that makes it easy to build simple, reliable, and efficient software.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="guile" class="soft_anch"><code>guile</code></a></td>
<td align="left">-&nbsp;<code>2.0.11</code><br/>-&nbsp;<code>2.2.2</code></td>
<td align="left"><a href="https://www.gnu.org/software/guile/">Website</a></td>
<td align="left">GNU Guile is the preferred extension system for the GNU Project, which features an implementation of the Scheme programming language.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="java" class="soft_anch"><code>java</code></a></td>
<td align="left">-&nbsp;<code>1.8.0_131</code></td>
<td align="left"><a href="http://www.oracle.com/technetwork/java/index.html">Website</a></td>
<td align="left">Java is a general-purpose computer programming language that is concurrent, class-based, object-oriented,[14] and specifically designed to have as few implementation dependencies as possible.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="julia" class="soft_anch"><code>julia</code></a></td>
<td align="left">-&nbsp;<code>0.6.4</code><br/>-&nbsp;<code>1.0.0</code><br/>-&nbsp;<code>1.1.0</code></td>
<td align="left"><a href="https://julialang.org">Website</a></td>
<td align="left">Julia is a high-level, high-performance dynamic programming language for numerical computing. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/julia">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="lua" class="soft_anch"><code>lua</code></a></td>
<td align="left">-&nbsp;<code>5.3.4</code></td>
<td align="left"><a href="https://www.lua.org">Website</a></td>
<td align="left">Lua is a powerful, efficient, lightweight, embeddable scripting language. It supports procedural programming, object-oriented programming, functional programming, data-driven programming, and data description.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="luarocks" class="soft_anch"><code>luarocks</code></a></td>
<td align="left">-&nbsp;<code>2.4.3</code></td>
<td align="left"><a href="https://luarocks.org/">Website</a></td>
<td align="left">LuaRocks is the package manager for Lua modules.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="manticore" class="soft_anch"><code>manticore</code></a></td>
<td align="left">-&nbsp;<code>20180301</code></td>
<td align="left"><a href="http://manticore.cs.uchicago.edu/">Website</a></td>
<td align="left">Manticore is a high-level parallel programming language aimed at general-purpose applications running on multi-core processors.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="nodejs" class="soft_anch"><code>nodejs</code></a></td>
<td align="left">-&nbsp;<code>8.9.4</code><br/>-&nbsp;<code>9.5.0</code></td>
<td align="left"><a href="https://nodejs.org/">Website</a></td>
<td align="left">Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. It provides the npm package manager.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="perl" class="soft_anch"><code>perl</code></a></td>
<td align="left">-&nbsp;<code>5.26.0</code></td>
<td align="left"><a href="https://www.perl.org">Website</a></td>
<td align="left">Perl 5 is a highly capable, feature-rich programming language with over 29 years of development. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/perl">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="php" class="soft_anch"><code>php</code></a></td>
<td align="left">-&nbsp;<code>7.3.0</code></td>
<td align="left"><a href="http://www.php.net">Website</a></td>
<td align="left">PHP (recursive acronym for PHP: Hypertext Preprocessor) is an open source general-purpose scripting language that is especially suited for web development.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="py-cython" class="soft_anch"><code>py-cython</code></a></td>
<td align="left">-&nbsp;<code>0.27.3_py36</code><br/>-&nbsp;<code>0.27.3_py27</code></td>
<td align="left"><a href="http://cython.org/">Website</a></td>
<td align="left">Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex).</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="py-ipython" class="soft_anch"><code>py-ipython</code></a></td>
<td align="left">-&nbsp;<code>6.1.0_py36</code><br/>-&nbsp;<code>5.4.1_py27</code></td>
<td align="left"><a href="https://ipython.org">Website</a></td>
<td align="left">IPython is a command shell for interactive computing in multiple programming languages, originally developed for the Python programming language.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="py-jupyter" class="soft_anch"><code>py-jupyter</code></a></td>
<td align="left">-&nbsp;<code>1.0.0_py36</code><br/>-&nbsp;<code>1.0.0_py27</code></td>
<td align="left"><a href="https://jupyter.org">Website</a></td>
<td align="left">Jupyter is a browser-based interactive notebook for programming, mathematics, and data science. It supports a number of languages via plugins.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="python" class="soft_anch"><code>python</code></a></td>
<td align="left">-&nbsp;<code>3.6.1</code><br/>-&nbsp;<code>2.7.13</code></td>
<td align="left"><a href="https://www.python.org">Website</a></td>
<td align="left">Python is an interpreted, interactive, object-oriented programming language. <br/><a href="https://www.sherlock.stanford.edu/docs/software/using/python">Usage on Sherlock</a></td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="ruby" class="soft_anch"><code>ruby</code></a></td>
<td align="left">-&nbsp;<code>2.4.1</code></td>
<td align="left"><a href="https://www.ruby-lang.org">Website</a></td>
<td align="left">A dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="rust" class="soft_anch"><code>rust</code></a></td>
<td align="left">-&nbsp;<code>1.35.0</code></td>
<td align="left"><a href="https://www.rust-lang.org/">Website</a></td>
<td align="left">The GNU Compiler Collection includes front ends for C, C++, Fortran, Java, and Go, as well as libraries for these languages (libstdc++, libgcj,...).</td>
</tr>
<tr>
<td align="left"><strong>language</strong></td>
<td align="left"><a id="scala" class="soft_anch"><code>scala</code></a></td>
<td align="left">-&nbsp;<code>2.12.6</code></td>
<td align="left"><a href="https://www.scala-lang.org">Website</a></td>
<td align="left">Scala combines object-oriented and functional programming in one concise, high-level language.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="ant" class="soft_anch"><code>ant</code></a></td>
<td align="left">-&nbsp;<code>1.10.1</code></td>
<td align="left"><a href="https://ant.apache.org/">Website</a></td>
<td align="left">Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="boost" class="soft_anch"><code>boost</code></a></td>
<td align="left">-&nbsp;<code>1.64.0</code><br/>-&nbsp;<code>1.69.0</code></td>
<td align="left"><a href="http://www.boost.org">Website</a></td>
<td align="left">Boost is a set of libraries for the C++ programming language that provide support for tasks and structures such as linear algebra, pseudorandom number generation, multithreading, image processing, regular expressions, and unit testing.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="cnmem" class="soft_anch"><code>cnmem</code></a></td>
<td align="left">-&nbsp;<code>1.0.0</code></td>
<td align="left"><a href="https://github.com/NVIDIA/ncmem">Website</a></td>
<td align="left">CNMeM is a simple library to help the Deep Learning frameworks manage CUDA memory.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="cub" class="soft_anch"><code>cub</code></a></td>
<td align="left">-&nbsp;<code>1.7.3</code></td>
<td align="left"><a href="https://nvlabs.github.io/cub/">Website</a></td>
<td align="left">CUB is a flexible library of cooperative threadblock primitives and other utilities for CUDA kernel programming.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="cutlass" class="soft_anch"><code>cutlass</code></a></td>
<td align="left">-&nbsp;<code>0.1.0</code></td>
<td align="left"><a href="https://github.com/NVIDIA/cutlass">Website</a></td>
<td align="left">CUTLASS is a collection of CUDA C++ template abstractions for implementing high-performance matrix-multiplication (GEMM) at all levels and scales within CUDA.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="eigen" class="soft_anch"><code>eigen</code></a></td>
<td align="left">-&nbsp;<code>3.3.3</code></td>
<td align="left"><a href="http://eigen.tuxfamily.org">Website</a></td>
<td align="left">Eigen is a C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="libctl" class="soft_anch"><code>libctl</code></a></td>
<td align="left">-&nbsp;<code>3.2.2</code><br/>-&nbsp;<code>4.0.1</code></td>
<td align="left"><a href="https://github.com/stevengj/libctl">Website</a></td>
<td align="left">libctl is a library for supporting flexible control files in scientific simulations.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="libgpuarray" class="soft_anch"><code>libgpuarray</code></a></td>
<td align="left">-&nbsp;<code>0.7.5</code></td>
<td align="left"><a href="http://deeplearning.net/software/libgpuarray">Website</a></td>
<td align="left">Library to manipulate tensors on the GPU.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="nccl" class="soft_anch"><code>nccl</code></a></td>
<td align="left">-&nbsp;<code>1.3.4</code><br/>-&nbsp;<code>2.0.4</code><br/>-&nbsp;<code>2.1.15</code><br/>-&nbsp;<code>2.2.13</code><br/>-&nbsp;<code>2.3.7</code></td>
<td align="left"><a href="https://github.com/NVIDIA/nccl">Website</a></td>
<td align="left">NCCL (pronounced 'Nickel') is a stand-alone library of standard collective communication routines, such as all-gather, reduce, broadcast, etc., that have been optimized to achieve high bandwidth over PCIe.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="opencv" class="soft_anch"><code>opencv</code></a></td>
<td align="left">-&nbsp;<code>3.3.0</code></td>
<td align="left"><a href="https://opencv.org">Website</a></td>
<td align="left">OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="petsc" class="soft_anch"><code>petsc</code></a></td>
<td align="left">-&nbsp;<code>3.10.3</code></td>
<td align="left"><a href="https://www.mcs.anl.gov/petsc/index.html">Website</a></td>
<td align="left">PETSc is a suite of data structures and routines for the scalable (parallel) solution of scientific applications modeled by partial differential equations.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="py-h5py" class="soft_anch"><code>py-h5py</code></a></td>
<td align="left">-&nbsp;<code>2.8.0_py36</code><br/>-&nbsp;<code>2.7.1_py27</code></td>
<td align="left"><a href="http://www.h5py.org">Website</a></td>
<td align="left">The h5py package is a Pythonic interface to the HDF5 binary data format.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="py-netcdf4" class="soft_anch"><code>py-netcdf4</code></a></td>
<td align="left">-&nbsp;<code>1.3.1_py36</code><br/>-&nbsp;<code>1.3.1_py27</code></td>
<td align="left"><a href="https://unidata.github.io/netcdf4-python/">Website</a></td>
<td align="left">netcdf4-python is a Python interface to the netCDF C library.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="py-numba" class="soft_anch"><code>py-numba</code></a></td>
<td align="left">-&nbsp;<code>0.35.0_py36</code><br/>-&nbsp;<code>0.35.0_py27</code></td>
<td align="left"><a href="http://numba.pydata.org">Website</a></td>
<td align="left">Numba is a compiler for Python array and numerical functions that gives you the power to speed up your applications with high performance functions written directly in Python..</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="py-pycuda" class="soft_anch"><code>py-pycuda</code></a></td>
<td align="left">-&nbsp;<code>2017.1.1_py27</code></td>
<td align="left"><a href="https://mathema.tician.de/software/pycuda/">Website</a></td>
<td align="left">PyCUDA lets you access Nvidias CUDA parallel computation API from Python.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="py-scikit-image" class="soft_anch"><code>py-scikit-image</code></a></td>
<td align="left">-&nbsp;<code>0.13.0_py27</code><br/>-&nbsp;<code>0.14.0_py27</code><br/>-&nbsp;<code>0.15.0_py27</code><br/>-&nbsp;<code>0.15.0_py36</code></td>
<td align="left"><a href="http://scikit-image.org/">Website</a></td>
<td align="left">scikit-image is a collection of algorithms for image processing.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="rabbitmq" class="soft_anch"><code>rabbitmq</code></a></td>
<td align="left">-&nbsp;<code>3.7.13</code></td>
<td align="left"><a href="http://rabbitmq.com">Website</a></td>
<td align="left">RabbitMQ is an open-source message broker.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="swig" class="soft_anch"><code>swig</code></a></td>
<td align="left">-&nbsp;<code>3.0.12</code></td>
<td align="left"><a href="http://www.swig.org">Website</a></td>
<td align="left">SWIG is an interface compiler that connects programs written in C and C++ with scripting languages such as Perl, Python, Ruby, and Tcl.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="tbb" class="soft_anch"><code>tbb</code></a></td>
<td align="left">-&nbsp;<code>2017.u2</code><br/>-&nbsp;<code>2018.u1</code><br/>-&nbsp;<code>2018</code><br/>-&nbsp;<code>2019</code></td>
<td align="left"><a href="https://software.intel.com/en-us/intel-tbb">Website</a></td>
<td align="left">Intel Threading Building Blocks (Intel TBB) is a widely used C++ library for shared-memory parallel programming and heterogeneous computing (intra-node distributed memory programming).</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="trilinos" class="soft_anch"><code>trilinos</code></a></td>
<td align="left">-&nbsp;<code>12.12.1</code></td>
<td align="left"><a href="https://trilinos.org/">Website</a></td>
<td align="left">Trilinos is a collection of open-source software libraries, called packages, intended to be used as building blocks for the development of scientific applications.</td>
</tr>
<tr>
<td align="left"><strong>lib</strong></td>
<td align="left"><a id="zeromq" class="soft_anch"><code>zeromq</code></a></td>
<td align="left">-&nbsp;<code>4.2.2</code></td>
<td align="left"><a href="http://zeromq.org">Website</a></td>
<td align="left">ZeroMQ (also spelled MQ, 0MQ or ZMQ) is a high-performance asynchronous messaging library, aimed at use in distributed or concurrent applications.</td>
</tr>
<tr>
<td align="left"><strong>mpi</strong></td>
<td align="left"><a id="impi" class="soft_anch"><code>impi</code></a></td>
<td align="left">-&nbsp;<code>2017.u2</code><br/>-&nbsp;<code>2018.u1</code><br/>-&nbsp;<code>2018</code><br/>-&nbsp;<code>2019</code></td>
<td align="left"><a href="https://software.intel.com/en-us/intel-mpi-library">Website</a></td>
<td align="left">Intel MPI Library is a multi-fabric message passing library that implements the Message Passing Interface, version 3.1 (MPI-3.1) specification.</td>
</tr>
<tr>
<td align="left"><strong>mpi</strong></td>
<td align="left"><a id="openmpi" class="soft_anch"><code>openmpi</code></a></td>
<td align="left">-&nbsp;<code>2.0.2</code><br/>-&nbsp;<code>2.1.1</code><br/>-&nbsp;<code>3.0.1</code><br/>-&nbsp;<code>4.0.0</code><br/>-&nbsp;<code>4.0.1</code><br/>-&nbsp;<code>3.1.2</code></td>
<td align="left"><a href="https://www.openmpi.org">Website</a></td>
<td align="left">The Open MPI Project is an open source Message Passing Interface implementation that is developed and maintained by a consortium of academic, research, and industry partners.</td>
</tr>
<tr>
<td align="left"><strong>mpi</strong></td>
<td align="left"><a id="py-mpi4py" class="soft_anch"><code>py-mpi4py</code></a></td>
<td align="left">-&nbsp;<code>3.0.0_py27</code></td>
<td align="left"><a href="https://bitbucket.org/mpi4py/mpi4py">Website</a></td>
<td align="left">MPI for Python provides Python bindings for the Message Passing Interface (MPI) standard. It is implemented on top of the MPI-&frac12;/3 specification and exposes an API which grounds on the standard MPI-2 C++ bindings.</td>
</tr>
<tr>
<td align="left"><strong>networking</strong></td>
<td align="left"><a id="gasnet" class="soft_anch"><code>gasnet</code></a></td>
<td align="left">-&nbsp;<code>1.30.0</code></td>
<td align="left"><a href="https://gasnet.lbl.gov">Website</a></td>
<td align="left">GASNet is a language-independent, low-level networking layer that provides network-independent, high-performance communication primitives tailored for implementing parallel global address space SPMD languages and libraries.</td>
</tr>
<tr>
<td align="left"><strong>networking</strong></td>
<td align="left"><a id="libfabric" class="soft_anch"><code>libfabric</code></a></td>
<td align="left">-&nbsp;<code>1.6.0</code><br/>-&nbsp;<code>1.6.2</code><br/>-&nbsp;<code>1.7.1</code></td>
<td align="left"><a href="https://ofiwg.github.io/libfabric/">Website</a></td>
<td align="left">The Open Fabrics Interfaces (OFI) is a framework focused on exporting fabric communication services to applications. Libfabric is the library that defines and exports the user-space API of OFI.</td>
</tr>
<tr>
<td align="left"><strong>networking</strong></td>
<td align="left"><a id="ucx" class="soft_anch"><code>ucx</code></a></td>
<td align="left">-&nbsp;<code>1.2.1</code><br/>-&nbsp;<code>1.4.0</code><br/>-&nbsp;<code>1.3.1</code></td>
<td align="left"><a href="http://www.openucx.org">Website</a></td>
<td align="left">UCX is a communication library implementing high-performance messaging for MPI/PGAS frameworks.</td>
</tr>
<tr>
<td align="left"><strong>parser</strong></td>
<td align="left"><a id="xerces-c" class="soft_anch"><code>xerces-c</code></a></td>
<td align="left">-&nbsp;<code>3.2.1</code></td>
<td align="left"><a href="https://xerces.apache.org/xerces-c/index.html">Website</a></td>
<td align="left">Xerces-C++ is a validating XML parser written in a portable subset of C++.</td>
</tr>
<tr>
<td align="left"><strong>sdk</strong></td>
<td align="left"><a id="google-cloud-sdk" class="soft_anch"><code>google-cloud-sdk</code></a></td>
<td align="left">-&nbsp;<code>240.0.0</code></td>
<td align="left"><a href="https://cloud.google.com/sdk">Website</a></td>
<td align="left">Command-line interface for Google Cloud Platform products and services.</td>
</tr>
</tbody>
</table></article></div></div></main><footer class="md-footer"><div class="md-footer-meta md-typeset"><div class="md-footer-meta__inner md-grid"><div class="md-footer-copyright"><div class="md-footer-copyright__highlight">Copyright &copy; 2014 - 2018 &mdash; <a href='mailto:srcc-support@stanford.edu'>Stanford Research Computing Center</a></div> Made with <i class="fa fa-heart md-highlight"></i> in <span class="md-footer-copyright__highlight"><a href='https://www.google.com/maps/place/Stanford,+CA'>Stanford, California</a> </span></div><div class="md-footer-social"><link rel="stylesheet" href="../../../assets/fonts/font-awesome.css"> <a href="https://srcc.stanford.edu" class="md-footer-social__link fa fa-globe"></a>  <a href="https://github.com/stanford-rc" class="md-footer-social__link fa fa-github"></a>  <a href="https://twitter.com/StanfordCompute" class="md-footer-social__link fa fa-twitter"></a>  <a href="https://www.linkedin.com/school/1792/" class="md-footer-social__link fa fa-linkedin"></a> </div></div></div></footer></div><script src="../../../assets/javascripts/application.9e29383c.js"></script><script>app.initialize({version:"1.0.4",url:{base:"../../.."}})</script><script src="../../.."></script><script src="../../../search/main.js"></script></body></html>